{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2022-11-28T13:03:16.839970Z",
     "start_time": "2022-11-28T13:03:16.301427Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "%pylab is deprecated, use %matplotlib inline and import the required libraries.\n",
      "Populating the interactive namespace from numpy and matplotlib\n"
     ]
    }
   ],
   "source": [
    "%load_ext sql\n",
    "%pylab inline\n",
    "plt.style.use('tableau-colorblind10')\n",
    "plt.rcParams['font.size'] = '16'\n",
    "\n",
    "figdir = '/home/glandrum/Papers/MolecularTimeSeries/images/'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Get assays with an associated document and pchembl_values from the local ChEMBL30 install"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2022-11-28T13:03:19.725682Z",
     "start_time": "2022-11-28T13:03:17.179502Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Done.\n",
      "248200 rows affected.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "%sql postgresql://localhost/chembl_30 \\\n",
    "     drop table if exists IC50_assays\n",
    "\n",
    "%sql postgresql://localhost/chembl_30 \\\n",
    "select assay_id,assays.chembl_id assay_chembl_id,description,tid,targets.chembl_id target_chembl_id, \\\n",
    "    count(distinct(molregno)) cnt,pref_name \\\n",
    "into temporary table IC50_assays \\\n",
    "          from activities \\\n",
    "          join docs using (doc_id) \\\n",
    "          join assays using(assay_id) \\\n",
    "          join target_dictionary as targets using (tid)\\\n",
    "          where pchembl_value is not null  \\\n",
    "          and docs.year is not null \\\n",
    "          and standard_units = 'nM' \\\n",
    "          and data_validity_comment is null \\\n",
    "          group by (assay_id,assays.chembl_id,description,tid,targets.chembl_id,pref_name) order by cnt desc"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Limit to those with 300-1000 datapoints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2022-11-28T13:03:19.974383Z",
     "start_time": "2022-11-28T13:03:19.728855Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " * postgresql://localhost/chembl_30\n",
      "132 rows affected.\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAfcAAAHqCAYAAAAZPQdiAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjUuMiwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy8qNh9FAAAACXBIWXMAAAsTAAALEwEAmpwYAAAnd0lEQVR4nO3deZhkVX3/8fcHBlDjAijgQmA0GA3GYOJoNG4DURFRMIoriMQFN4hJNFEkKiIJPjGKUeJCokJ+jBtxg6igBhzRDOpI1IAKiqCiEAYHUGSRge/vj3tbi6K6u7qnehpPv1/PU09NnXvOvadO1fSn7p6qQpIktWOzxe6AJEmaLMNdkqTGGO6SJDXGcJckqTGGuyRJjTHcJUlqzLLF7sCk3OUud6nly5cvdjckSdokvva1r11eVduNmtZMuC9fvpy1a9cudjckSdokkvxgumlulpckqTGGuyRJjTHcJUlqjOEuSVJjDHdJkhpjuEuS1BjDXZKkxhjukiQ1xnCXJKkxhrskSY0x3CVJaozhLklSYwx3SZIaY7hLktQYw12SpMYY7pIkNcZwlySpMYa7JEmNMdwlSWrMssXuwK1VDlk10fnVsftPdH6SJE3HNXdJkhpjuEuS1BjDXZKkxhjukiQ1xnCXJKkxhrskSY0x3CVJaozhLklSYwx3SZIaY7hLktQYw12SpMYY7pIkNcZwlySpMYa7JEmNMdwlSWqM4S5JUmMMd0mSGmO4S5LUGMNdkqTGGO6SJDXGcJckqTGGuyRJjTHcJUlqjOEuSVJjDHdJkhpjuEuS1Jixwj3JjknenmRNkmuSVJLlQ3WO78tHPb4zxjIumqbtk+b31iRJWpqWjVlvF+BpwNeAM4HHjqjzBuBdQ2XLgQ8AJ4+5nNOAI4bKzhuzrSRJYvxw/0JV7QCQ5PmMCPequgC4YLAsyWP6f54w5nIur6qzxqwrSZJGGGuzfFXdNM/5Hwh8rarOnWd7SZI0Rwt2QF2Sh9Ftzh93rR3gif0+/euTnOX+dkmS5m4hj5Y/ELiBbp/7OE4BDgX2BPYHrgM+luSA6RokOTjJ2iRr161bt7H9lSSpCePuc5+TJFvRHYD3n1V1+ThtqurQoXl8DDgLOBo4cZo2xwHHAaxYsaI2ps+SJLViodbc9wW2Zm6b5G+mqm4ETgJ2THK3CfVLkqTmLVS4Pwe4HPjURs4n/bNr5ZIkjWni4Z5kB7pT5d5fVTdsxHyWAU8FflhVl06qf5IktW7sfe5J9uv/+cD+ea8k64B1VbV6oOr+/Xyn3SSfZANwQlU9r3/9TLpN+Z8CfgTsALy0X9Yzx+2jJEma2wF1Jw29fkf/vBpYOVD+HOCcqjp7hnlt3j+mXAhsD7wJ2Ba4Bvgq8LiqOm0OfZQkackbO9yrKrPXgqraba7z6q9Kt8e4fZEkSdPzrnCSJDXGcJckqTGGuyRJjTHcJUlqjOEuSVJjDHdJkhpjuEuS1BjDXZKkxhjukiQ1xnCXJKkxhrskSY0x3CVJaozhLklSYwx3SZIaY7hLktQYw12SpMYY7pIkNcZwlySpMYa7JEmNMdwlSWqM4S5JUmMMd0mSGmO4S5LUGMNdkqTGGO6SJDVm2WJ3QPOXQ1ZNdH517P4TnZ8kaXG45i5JUmMMd0mSGmO4S5LUGMNdkqTGGO6SJDXGcJckqTGGuyRJjTHcJUlqjOEuSVJjDHdJkhpjuEuS1BjDXZKkxhjukiQ1xnCXJKkxhrskSY0ZK9yT7Jjk7UnWJLkmSSVZPlRneV8+6rH1GMvYLMlhSS5Kcl2SbyR5yvzeliRJS9e4a+67AE8DrgDOnKXu0cBDhx4/H2MZbwCOAI4F9gLOAk5K8vgx+yhJkoBlY9b7QlXtAJDk+cBjZ6j7/ao6ay6dSLI98ArgjVX1T33xGUl2Ad4IfGou85MkaSkba829qm5a4H7sCWwJnDhUfiJw/yT3XODlS5LUjIU4oO7oJBuSXJXk5CT3H6PN/YDrge8NlZ/bP+860R5KktSwcTfLj+N64N3AZ4B1wH2BVwP/neTBVfXtGdpuC1xZVTVUvn5guiRJGsPEwr2qLgFeNFB0ZpJT6da+DwcOmKF5gOFgnyqfvlFyMHAwwE477TSn/kqS1KoFPc+9qn4EfBF40CxV1wPbJBkO820Gpo+a/3FVtaKqVmy33XYb11lJkhqxKS5iM91a+aBzga2A3xkqn9rX/q1Jd0qSpFYtaLgn2Ql4GPDlWaqeCvwS2H+o/ADgnKq6cAG6J0lSk8be555kv/6fD+yf90qyDlhXVauTvJnux8IaugPq7gMcBtwE/MPQvDYAJ1TV8wCq6rIkxwCHJfk5cDbwdGAPYN/5vjlJkpaiuRxQd9LQ63f0z6uBlXSb1l8MHATcAbgcOB14fVWdN9R28/4x6HDgauBlwF2B84CnVdUpc+ijJElL3tjhXlUzHrleVe8F3jvfeVXVjcBR/UOSJM2Td4WTJKkxhrskSY0x3CVJaozhLklSYwx3SZIaY7hLktQYw12SpMYY7pIkNcZwlySpMYa7JEmNMdwlSWqM4S5JUmMMd0mSGmO4S5LUGMNdkqTGGO6SJDXGcJckqTGGuyRJjTHcJUlqjOEuSVJjDHdJkhpjuEuS1BjDXZKkxhjukiQ1xnCXJKkxhrskSY0x3CVJaozhLklSYwx3SZIaY7hLktQYw12SpMYY7pIkNcZwlySpMYa7JEmNMdwlSWqM4S5JUmOWLXYHloocsmqxuyBJWiJcc5ckqTGGuyRJjTHcJUlqjOEuSVJjDHdJkhozVrgn2THJ25OsSXJNkkqyfKjOnyY5MckFSa7tn9+ZZPsxl3FRP9/hx5Pm/rYkSVq6xj0VbhfgacDXgDOBx46o8yLg9sBRwPeBewOvB/ZM8gdVdfUYyzkNOGKo7Lwx+yhJkhg/3L9QVTsAJHk+o8P9JVW1buD16iTnA6vpfhi8d4zlXF5VZ43ZJ0mSNMJYm+Wr6qYx6qwbUfzV/vkec+mUJEmav4U+oO5R/fO3x6z/xH6f/vVJznJ/uyRJc7dg4Z7kDsBb6YL942M0OQU4FNgT2B+4DvhYkgNmWMbBSdYmWbtu3agNB5IkLT0Lcm35JMuAD9Btjn9YVW2YrU1VHTo0j48BZwFHAydO0+Y44DiAFStW1EZ2W5KkJkx8zT3JZsAJwKOBJ1XVN+czn6q6ETgJ2DHJ3SbYRUmSmrYQa+7vAp4O7FdV/7WR80r/7Fq5JEljmmi4J3kz8HzgOVX18Y2c1zLgqcAPq+rSCXRPkqQlYexwT7Jf/88H9s97JVkHrKuq1UleCfw13fns303ykIHm66rqgoF5bQBOqKrn9a+fCewLfAr4EbAD8NJ+Wc+c1zuTJGmJmsua+0lDr9/RP68GVgJ79a+f2z8GnQAcNPB68/4x5UJge+BNwLbANXTnyD+uqk6bQx8lSVryxg73qsos01fOd179Ven2GLe9JEmanneFkySpMYa7JEmNMdwlSWrMglyhTloIOWTVxOdZx+4/8XlK0mJzzV2SpMYY7pIkNcZwlySpMYa7JEmNMdwlSWqM4S5JUmMMd0mSGmO4S5LUGMNdkqTGGO6SJDXGcJckqTGGuyRJjTHcJUlqjOEuSVJjDHdJkhpjuEuS1BjDXZKkxhjukiQ1xnCXJKkxhrskSY0x3CVJasyyxe6Abj1yyKqJzq+O3X+i85Mkjcc1d0mSGmO4S5LUGMNdkqTGGO6SJDXGcJckqTGGuyRJjTHcJUlqjOEuSVJjDHdJkhpjuEuS1BjDXZKkxhjukiQ1xnCXJKkxhrskSY0x3CVJasxY4Z5kxyRvT7ImyTVJKsnyEfW2SfJvSS5P8oskn0ty/zGXsVmSw5JclOS6JN9I8pQ5vh9Jkpa8cdfcdwGeBlwBnDmqQpIAJwOPAw4FngJsAZyRZMcxlvEG4AjgWGAv4CzgpCSPH7OPkiQJWDZmvS9U1Q4ASZ4PPHZEnX2AhwN7VNUZfd01wIXA3wJ/Md3Mk2wPvAJ4Y1X9U198RpJdgDcCnxqzn5IkLXljrblX1U1jVNsH+MlUsPftrgJOAfadpe2ewJbAiUPlJwL3T3LPcfopSZIme0Dd/YBzRpSfC+yU5PaztL0e+N6ItgC7bnz3JElaGiYZ7tvS7ZMftr5/3maWtldWVU3TdtuN7JskSUvGJMM9wHA4T5UvSNskBydZm2TtunXrxliMJEntm2S4r2f0GvbUGvuotfrBttv0R9yParueEarquKpaUVUrtttuuzl1VpKkVk0y3M+l23c+bFfgh1V19SxttwJ+Z0RbgG9tfPckSVoaJhnuJwP3SPKoqYIkdwSe2E+byanAL4H9h8oPAM6pqgsn2E9Jkpo27nnuJNmv/+cD++e9kqwD1lXVaroAXwOcmORv6DbDH0a33/wfh+a1ATihqp4HUFWXJTkGOCzJz4GzgacDezD7aXSSJGnA2OEOnDT0+h3982pgZVXdlOQJwD/1025DF/a7V9WPhtpu3j8GHQ5cDbwMuCtwHvC0qjplDn2UJGnJGzvcq2rWo96raj3w3P4xp3lV1Y3AUf1DkiTNk3eFkySpMYa7JEmNMdwlSWqM4S5JUmMMd0mSGmO4S5LUGMNdkqTGGO6SJDXGcJckqTGGuyRJjTHcJUlqjOEuSVJjDHdJkhpjuEuS1BjDXZKkxox9P3dJ2hRyyKqJzq+O3X+i85N+E7jmLklSYwx3SZIaY7hLktQYw12SpMYY7pIkNcZwlySpMYa7JEmNMdwlSWqM4S5JUmMMd0mSGmO4S5LUGMNdkqTGGO6SJDXGcJckqTGGuyRJjTHcJUlqjOEuSVJjDHdJkhpjuEuS1BjDXZKkxixb7A5I0kLKIasmPs86dv+Jz1OaJNfcJUlqjOEuSVJjDHdJkhpjuEuS1BjDXZKkxkw03JN8PklN8zh1lrbTtXvAJPsoSVLrJn0q3EuAOw6VPRR4C3DyGO2PB949VHb+xndLkqSlY6LhXlXfGi5L8gLgl8AHx5jFj6vqrEn2SZKkpWZB97knuS3wVOCUqlq/kMuSJEmdhT6g7snAHYATxqz/4iTXJ7kmyelJHrGAfZMkqUkLHe4HApcBnx6j7ol0++wfDRwM3Bk4PcnK6RokOTjJ2iRr161bt/G9lSSpAQsW7knuThfUq6pqw2z1q+rZVfWhqjqzqk4EHg78BDhqhjbHVdWKqlqx3XbbTazvkiT9JlvINfcD+vmPu0n+Zqrq58AngQdNslOSJLVuIcP9QOAbVfWNjZhHgJpQfyRJWhIWJNyTrADuxzzX2vt53BHYG/jypPolSdJSsFD3cz8Q2AC8f3hCkp2BC4Ajq+rIvuwVwH2AM+j2s+8MvAK4K+CNkyVJmoOJh3uSLYBnAqdW1f+NqgJszs23GpwH/Fn/uBPwM+BLwPOq6iuT7qMkSS2beLhX1Q3AtIeuV9VFdAE/WHYKcMqk+yJJ0lLkXeEkSWqM4S5JUmMMd0mSGrNQR8tL5JBVi92FWU26j3WsJ3dIWnyuuUuS1BjDXZKkxhjukiQ1xnCXJKkxhrskSY0x3CVJaozhLklSYwx3SZIaY7hLktQYw12SpMYY7pIkNcZwlySpMYa7JEmNMdwlSWqM4S5JUmMMd0mSGmO4S5LUGMNdkqTGGO6SJDXGcJckqTGGuyRJjVm22B2Q9Jsth6xa7C5IGuKauyRJjTHcJUlqjOEuSVJjDHdJkhpjuEuS1BjDXZKkxhjukiQ1xnCXJKkxhrskSY0x3CVJaozhLklSYwx3SZIaY7hLktQYw12SpMYY7pIkNWai4Z5kZZIa8bhyjLa3SfKmJJckuTbJmiSPnGT/JElaCpYt0Hz/AvjqwOsNY7R5D7A38DfA94GXAqcleWhVfX3iPZQkqVELFe7frqqzxq2cZDfgWcBzq+p9fdlq4FzgSGCfBemlJEkNurXsc98HuAH40FRBVW0APgjsmWSrxeqYJEm/aRYq3FcluTHJT5O8P8lOs9S/H3BhVV0zVH4usCWwy4L0UpKkBk16s/xVwJuB1cDPgD8EXg2sSfKHVXXZNO22Ba4YUb5+YLokSRrDRMO9qv4H+J+BotVJvgB8he4gu7+bpmmAmqZ8WkkOBg4G2Gmn2TYOSJK0NCz4PveqOhs4H3jQDNXWM3rtfJuB6aPmfVxVraiqFdttt93GdVSSpEZsqgPqplszn3IucM8ktxsq3xX4JfC9heqYJEmtWfBwT7IC+F3gyzNUOxnYAnjqQLtlwNOBz1TV9QvaSUmSGjLRfe5JVgEXAmcDV9IdUHcY8GPg7X2dnYELgCOr6kiAqvp6kg8Bb02yRT+PFwP3BPafZB8lSWrdpI+WPwd4JnAocDvgUuCjwOuq6vK+ToDNueVWgz8H/h44Ctga+AbwuH6fvSRJGtOkj5Y/Gjh6ljoXMeIo+Kq6Fvjr/iFJkubp1nKFOkmSNCGGuyRJjTHcJUlqjOEuSVJjDHdJkhpjuEuS1BjDXZKkxhjukiQ1xnCXJKkxhrskSY0x3CVJaozhLklSYwx3SZIaY7hLktQYw12SpMZM9H7u0lKXQ1ZNdH517P4TnZ8mw89Zt3auuUuS1BjDXZKkxhjukiQ1xnCXJKkxhrskSY0x3CVJaozhLklSYwx3SZIaY7hLktQYw12SpMYY7pIkNcZwlySpMYa7JEmNMdwlSWqM4S5JUmMMd0mSGmO4S5LUGMNdkqTGGO6SJDXGcJckqTHLFrsDkqaXQ1ZNfJ517P4Tn6c0V5P+bvu9vjnX3CVJaozhLklSYwx3SZIaY7hLktQYw12SpMZMLNyT7JfkI0l+kOTaJOclOTrJHcZoW9M8HjCp/kmStFRM8lS4VwA/BF4NXAz8IXAEsHuSP6mqm2Zpfzzw7qGy8yfYP0mSloRJhvsTq2rdwOvVSdYDJwArgdNnaf/jqjprgv2RJGlJmthm+aFgn/LV/vkek1qOJEma2UIfUPeo/vnbY9R9cZLrk1yT5PQkj1jIjkmS1KoFC/ck9wCOBD5XVWtnqX4i8BLg0cDBwJ2B05OsnGUZBydZm2TtunWjNhxIkrT0LMi15ZPcHvgEsAH489nqV9WzB16emeQTwDnAUcDDZ2h3HHAcwIoVK2pj+ixJUismvuae5DbAycC9gD2r6uK5zqOqfg58EnjQhLsnSVLzJrrmnmQL4CPAg4FHV9X/bszsANfGJUmao4mFe5LNgFXAnwJ7b8xpbUnuCOwNfHlC3ZMkacmY5Jr7vwBPBf4e+EWShwxMu7iqLk6yM3ABcGRVHQmQ5BXAfYAzgJ8AO9NdEOeugDfolSRpjiYZ7nv1z4f3j0Gvp7taXYDNufm+/vOAP+sfdwJ+BnwJeF5VfWWC/ZMkaUmYWLhX1fIx6lxEF/CDZacAp0yqH5IkLXXeFU6SpMYY7pIkNcZwlySpMQtyhTpJt145ZNVid0FDJv2Z1LGTP9Ho1v69WYj+LcQ4biquuUuS1BjDXZKkxhjukiQ1xnCXJKkxhrskSY0x3CVJaozhLklSYwx3SZIaY7hLktQYw12SpMYY7pIkNcZwlySpMYa7JEmNMdwlSWqM4S5JUmMMd0mSGmO4S5LUGMNdkqTGGO6SJDXGcJckqTGGuyRJjVm22B2QJE1WDlm12F1owqTHsY7df6Lzm4lr7pIkNcZwlySpMYa7JEmNMdwlSWqM4S5JUmMMd0mSGmO4S5LUGMNdkqTGGO6SJDXGcJckqTGGuyRJjTHcJUlqjOEuSVJjDHdJkhpjuEuS1JiJhnuS307yH0muSvKzJB9NstOYbW+T5E1JLklybZI1SR45yf5JkrQUTCzck9wOOB24L/Ac4NnAvYEzkvzWGLN4D/AC4LXAE4BLgNOSPGBSfZQkaSlYNsF5vQC4F3CfqvoeQJJvAt8FXgi8ZbqGSXYDngU8t6re15etBs4FjgT2mWA/JUlq2iQ3y+8DnDUV7ABVdSHwJWDfMdreAHxooO0G4IPAnkm2mmA/JUlq2iTD/X7AOSPKzwV2HaPthVV1zYi2WwK7bHz3JElaGiYZ7tsCV4woXw9ssxFtp6ZLkqQxTHKfO0CNKMsY7TKftkkOBg7uX16d5LwxljWuuwCXT3B+S5XjOBmO42Q4jpPjWM5R/uWAUcUbM447TzdhkuF+BaPXsLdh9Fr5oPXAqFPmthmYfgtVdRxw3LgdnIska6tqxULMeylxHCfDcZwMx3FyHMvJWKhxnORm+XPp9p0P2xX41hht79mfTjfc9pfA927ZRJIkjTLJcD8ZeEiSe00VJFkOPKyfNlvbLYCnDrRdBjwd+ExVXT/BfkqS1LRJhvu/AhcBn0iyb5J9gE8APwLePVUpyc5JNiR57VRZVX2d7jS4tyZ5fpI/pTsN7p7A6ybYx7lYkM39S5DjOBmO42Q4jpPjWE7Gwuxarhp1HNs8Z9ZdavYY4DF0B8P9F/CXVXXRQJ3lwIXA66vqiIHy2wJ/T3cxm62BbwCvrKrPT6yDkiQtARMNd0mStPiavytckj2TnJ7k0iTXJ7k4yYeT7DpUb5sk/5bk8iS/SPK5JPcfMT9vcNNLcmqSSnLUULljOY0kK/sxG35cOVTPMRxDkscn+UKSq/ubVa1NssfAdMdxFkk+P813spKcOlDPsZxFkocl+UySy/rv49lJnjtUZ9OMY1U1/QCeCbwJ2A94FN0Nbc4Ffgbs3NcJcCZwcV//ccBqunMPdxya3yrgSrpr6f8p8FHgWuABi/1eF2FcL6G7PsFRA+WO5czjtrIfs0OBhww8VjiGcx7LF9JdtnpqV+CewCuBJziOcxrHXYe+iw8B/qr/nr7EsRx7HP+gf59n0F1y/TF0x5sV8OJNPY6LPiCL9CHcpx/wl/ev9+1f7z5Q505059e/baBst77enw+ULQPOA05e7Pe1Ccdva+DS/ss5HO6O5cxjt7J/34+eoY5jOPs4Lu//0P2l47gg4/se4HpgW8dy7DH7B7pTt28/VH4WsGZTj2Pzm+Wn8dP++Yb+eR/gJ1V1xlSFqroKOIWb3/TGG9x0/hE4t6o+MGKaY7nxHMPZPRe4CXjXDHUcx3noD25+KnBKVU1dQMyxnN2WdO/92qHyK/n1LvBNNo5LJtyTbJ5kyyT3pttUcindQMHMN73ZKcntB+pdWEv4BjdJHg4cCLxkmiqO5XhWJbkxyU+TvD/dmSZTHMPZPRz4DvCMJBekO732e0leOlDHcZyfJwN3AE4YKHMsZ3d8//y2JHdPsnWSqU3qx/TTNtk4LplwB75Mt5npfLp9I3tU1WX9tNluXLPNmPWavsFNki3ofhj9U1VNdx1/x3JmVwFvBp4P7AG8AXg0sCbJ9n0dx3B2dwfuTXc8zRuBxwKfBY5N8rK+juM4PwcClwGfHihzLGdRVefQ7XbbF/gx3Tj8C/Ciqppakdxk4zjpG8fcmj0buCNwL+AVwGeTPLy6c/DHvXHNvG5w05BXAlPXI5iOYzmDqvof4H8GilYn+QLwFeAvgL/DMRzHZnRrlwdV1Uf7stPTXUfjsCRvw3GcsyR3p/ux+c/9ZuBfTcKxnFG/VfgjdGvXL6LbPL8v8K4k11XVKjbhOC6ZcK+qb/f//HKST9NdTe9VdB/Ceqa/6Q38+hfUvG5w04J+s/HhdGucWw3t89kqydbAz3Es56yqzk5yPvCgvsgxnN1P6dbcPztU/hm6I5DvhuM4HwfQ/XA6YajcsZzdP9DtJ39CVU0dz/VfSe4M/HOSD7AJx3EpbZb/laq6ku5mNFP7LWa66c0Pq+rqgXr3zNK8wc29gNsAJ9J9Aace0G0JuQK4P47lfA3+UncMZ3fuNOVTazY34TjOx4HAN6rqG0PljuXs7k83djcMlX8FuDOwPZtwHJdkuCfZAbgvcEFfdDJwjySPGqhzR+CJ3PymN0v5BjdfB3Yf8YAu8Hen+8I5lnOUZAXwu3THhYBjOI6P9c97DpXvCVxcVZfiOM5J/z28H7dcawfHchyXAg9IsuVQ+R8D19GtbW+6cVzscwMX+kH3R+A1dPs+dqe78MV36E5P+N2+zmbAf9Pd5OYZdH8gPt9/GL89NL8P0q2lPp/uKMj/6D+4P1rs97pI4zt8nrtjOfN4rQKOojsieQ/g5XQXsPghcBfHcOxxDHA63eb5F9EdUHdc/308yHGc15i+jW6z8g4jpjmWs4/ffv3377Q+bx4LHNuXvWVTj+OiD8gmGPBXAl+jC/Nr6C4C8G5g+VC9bYH39oN8Dd1Nb3YbMb/bAm+h+5V2Hd3a1srFfp+LOL43C3fHctbxOgz4Jt1R8zf0/8mPA+7mGM55LO9IdzTy/9Ftqvwm8CzHcV5juQWwju7c9unqOJazj+NefVivozsG6et0pw1vvqnH0RvHSJLUmCW5z12SpJYZ7pIkNcZwlySpMYa7JEmNMdwlSWqM4S5JUmMMdy2oJAclqSRXJtlmaNqyftoRi9CvI/pl36rvr5BksyRvTXJJkpuSfHwTLvuIJJ4r26gky/v/Awctdl80eYa7NpU70V1QSHOzH/AyulubPgz420247H8DHroJlydpQm7Vay1qymeAQ5O8tbrrfjcvyVa18dfS/r3++a1VddPG9mkuqupi4OJNuUxJk+GauzaVo/rnw2eqNN2m4CTHJ7lo4PXUJsUXJTk6yaVJfp7kxCS3S7JLktOSXJ3ke0meM80ify/JGUmu6Td9H5nkZv8vktwlyTuT/DjJ9Um+k+TgoTpTux8emeSkJFfy6xvBTPdeH5dkTZJrk1yV5ONJ7jMw/SLgiP7ljTNtQp1uE2uSlX35yoGyzyf5YpJHJzm7f+/nJHnSUNtbfBZJtkvy/iQ/63e1/HuSJ41YxkVJjh/Rz1vshkmyW5KTk1zRj8WXkjximmEbnt9uST6W5Kd92/OSHDYwPUn+qi//Zf8ZH9vfrGO4X0cleXmSHyT5RZJPJtm+f3y4/4x+lOSVQ20HP/uP99+5nyb5lyS3Hap7t37MLu+/S99McsBs496XT/d/4IX99/aS/jM5JcmOQ21vl+Qdfb+uTnIysOOIZTwoyWf7etck+X6Sd4zzWejWxXDXpnIJ3U0UDk6y8wTnexhwd+A5wGvp7pr0LrobBn0S+DO6a46/L8moWy1+HPgc8CTg/XQ3GXrt1MQ+BL4E7E0XtHsDpwDvTHLoiPmtAi6k25z+quk6neRxff+u7vv8YuD3gS8muUdf7c+A4/t/P7R/fHK6ec7R7wD/THft6ifTfT7/kWSXGVvBR4EnAK/u+70BePt8O5Hkj+hupLEt8ALgKXQ3g/lckgfO0vbBwJr+vfwV3WfzFm4eWn/fl32W7s5b/wgcBHwyQz/igGfT3cznJcChwCOAf6f7Ln2z79ungDcmefyILp1Id2fEJwPH9O/nnQP9/S1gNd31x19N9537X+D/ZejH4hwdRnf76ufS7cJ5KN33cNC76W5AMvV5n0f3ff+VJLenu+nJjXRj9HjgSNzC+5tpsS+076PtB90fiaL747Mt3Q183ttPW9ZPO2Kg/hHd1/IW8zkeuGjg9fK+7elD9T7alx8wULYNXQi9bng5wKuG2v8r3Q0ftu5fv4bupg33HlHvcmDZ0Ps8ZsxxWQt8d6p9X3ZPupvJvGWg7KhR4zFiflPjcdBQ+cq+fOVA2ef75dx7oGx7uj/qr57uswAe08/rGUPL+PSIZVwEHD+in8Of938B3wa2HCjbvC/7+Czv+Qt0N9653TTTt+0/u+OHyg/o+7HPUL/OH/o83tKX/91A2TLgMuB9I77j7xpazuH9mE7dffKQ4XHqyz/Xz3Pzef4fWD1U7xV9+d371/fp+zH8XX8nN7+L3or+9R+M8x32cet+uOauTaaq1gNvBg4c3Py8kT499Po7/fNpA8u9gu6P52+PaP/hodcfBG5PtxYN8Di6zesXpju6f1m6I+xPA+4M7DrU/mOzdbhfg/sj4ENVtWGgnxfSbSV41GzzmIDvVtV3B5Z9Gd0Y7TRDm4fShcRHhso/OJ8O9JusHwWcBNw0MLahC7xHztD2dnQHGK6qqmumqfYQYCu6Nerh/m7gluP82cHPg9HfpQ10a+fjfpc2Ax7cv34k8OOq+vxQvROB7bjld2lcw1tz/rd/nvos/7jvx6j+Dfou3Y/vdyc5IMmo96jfEIa7NrVj6G51eOSE5nfF0OtfzlB+mxHt/2+a11Obxren+6N8w9DjpH76nYfaXzJ7l9mGLsBG1b2Ubo1zoa0fUXY9o8doyt2AK6rqhqHy4TEc17Z0a+mv4ZbjewiwzYhN51O2ofv7NdMBf1PjeLNx7gP6p9xynBf6u7TtcF96lw5Mn4/hz3LqIM6pPt5tlv4BUFVXAbsDPwHeAfywPxbjKfPslxaR+1K0SVXV1UmOpluDf9OIKtcBJNmyqn45UD4copOyA/D9odcAP+6ff0q3RvuyadqfN/R6nPPCr+jr3XXEtLv2y5yr6/rnLYfKJzlul9AF7hZDAb/DiLrXDfclyXB4XQncRHdP9n8ftcCa/gyBK/q295hmOvw69O4KnDvQj2V04zKfcZ7JDoPL4ZbfpfV0m8iHTX0Ppvoz6f8DUz8opvuu/0pVfR14Sj9GK+j25384yW5Vdc48l69F4Jq7FsM76P7gHTVi2g/656nN4iTZGviTBerL04ZeP4PuILepP2SnAvcFflhVa0c8fj7XBVbVL4CvAU9NsvlUeX+g4Z/QHXQ1V/9Ht8b2+0Ple89jXtNZQ7emPbwm94wRdX8woi9PGHzRj8OZwG7A2aPGd7qO9JvivwgcMHxE+oCz6MZkuH9Pp1uxmc84z2TUd+km4Cv969XAjkkeNlTvWXQ/IL/dv570/4Ev9/0Y1b+RqmpDVZ1Ft1VlM359SqZ+Q7jmrk2uqq5PciRw3IjJnwauAv41yevo9pn+LV3gLoQX9Jt+vwrsSXdE8RFVdWU//Ri6MDgzyTF0a+q/RRf4j6iqfee53NfQ7Sv9z/5Uo9sDr6d772+e68yqqpJ8CHhekvP7fu5Nd0DdRFTVZ5N8kW6f7F3o9tE+nVuGOHT7c9/bj9l/0gX4QSPq/TXdgXGnJXkP3VrmXeiOSdi8qqY944DuwLHVwJokb6bbRH8v4AFVdWhVrU/yFuCwJL+gO9L99+h+VH6RyZ15MOXxSd5Ed02HBwOvA/69qs7vpx9PtwXoo0kO7/u7P92Bii+sqhv7ehP9P1BV5yV5P3DkwHf9MXRHw/9KkicAB9OdQXIh3ff8L+gOMF0zn2Vr8bjmrsXyPrpwuJk+VJ9At6bxYeBoulOtzligfuxL94fuZLqjqI8C3jDQn6vo1pg+RXeFvdOA9/bt5t2nqjqVLny3pnuf76Jbc3t4Vf1knrN9Gd3ZAkcAH6Lb5zrqdL2N8WS6sTi6X8Yyuv3jw06gC7cn0506uCfdqX03U1VnAw+i2yT9Nrpg/Gfg/nShP62q+irdQXU/ovuOfAr4G26+H/5wuh8Qe9H9yHgV3S6AvWfY5D9fBwC/S3dQ5cvpzqh4yUB/f0F3EN9ngDcCn6D70fPsqjpuoN6VTP7/wAuB99D9IPoY3Y/TZw3V+S5wLd0Pz0/T/R/dADymugsa6TdIqrx0tKT5S3fxmjOA3UccCd68dBcOeh/dqYXfW+TuSIBr7pIkNcdwlySpMW6WlySpMa65S5LUGMNdkqTGGO6SJDXGcJckqTGGuyRJjTHcJUlqzP8H/u10oV7P+pIAAAAASUVORK5CYII=\n",
      "text/plain": [
       "<Figure size 576x576 with 1 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "figsize(8,8)\n",
    "d = %sql select * from IC50_assays where cnt>=300 and cnt<=1000 order by cnt desc;\n",
    "df = d.DataFrame()\n",
    "hist(df.cnt,bins=20)\n",
    "xlabel('Number of unique compounds');\n",
    "\n",
    "plt.savefig(f'{figdir}/chembl_assays_compound_count.pdf',bbox_inches='tight')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2022-11-28T13:03:19.982994Z",
     "start_time": "2022-11-28T13:03:19.975895Z"
    },
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>assay_id</th>\n",
       "      <th>assay_chembl_id</th>\n",
       "      <th>description</th>\n",
       "      <th>tid</th>\n",
       "      <th>target_chembl_id</th>\n",
       "      <th>cnt</th>\n",
       "      <th>pref_name</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1641660</td>\n",
       "      <td>CHEMBL3888383</td>\n",
       "      <td>Cell Based Assay: TRKA co-expressed with p75: ...</td>\n",
       "      <td>11902</td>\n",
       "      <td>CHEMBL2815</td>\n",
       "      <td>794</td>\n",
       "      <td>Nerve growth factor receptor Trk-A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1528837</td>\n",
       "      <td>CHEMBL3705899</td>\n",
       "      <td>In Vitro Assay: Recombinant human factor D (ex...</td>\n",
       "      <td>105346</td>\n",
       "      <td>CHEMBL2176771</td>\n",
       "      <td>768</td>\n",
       "      <td>Complement factor D</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1527879</td>\n",
       "      <td>CHEMBL3705123</td>\n",
       "      <td>Inhibiton Assay: Inhibition assay using human ...</td>\n",
       "      <td>11904</td>\n",
       "      <td>CHEMBL2998</td>\n",
       "      <td>749</td>\n",
       "      <td>P2X purinoceptor 3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>1528786</td>\n",
       "      <td>CHEMBL3706373</td>\n",
       "      <td>Homogeneous Time Resolved Fluorescence Assay: ...</td>\n",
       "      <td>10102</td>\n",
       "      <td>CHEMBL4550</td>\n",
       "      <td>733</td>\n",
       "      <td>5-lipoxygenase activating protein</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>1641435</td>\n",
       "      <td>CHEMBL3888158</td>\n",
       "      <td>PARP Enzyme Inhibition Assay: PARP1 assay was ...</td>\n",
       "      <td>11663</td>\n",
       "      <td>CHEMBL3105</td>\n",
       "      <td>724</td>\n",
       "      <td>Poly [ADP-ribose] polymerase-1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>1528055</td>\n",
       "      <td>CHEMBL3705362</td>\n",
       "      <td>Mobility-Shift kinase assay: Compounds of the ...</td>\n",
       "      <td>10906</td>\n",
       "      <td>CHEMBL2599</td>\n",
       "      <td>720</td>\n",
       "      <td>Tyrosine-protein kinase SYK</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>1642374</td>\n",
       "      <td>CHEMBL3889097</td>\n",
       "      <td>Functional Assay: The functional activity of c...</td>\n",
       "      <td>103857</td>\n",
       "      <td>CHEMBL1667665</td>\n",
       "      <td>700</td>\n",
       "      <td>Transient receptor potential M8 protein</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>1638439</td>\n",
       "      <td>CHEMBL3881499</td>\n",
       "      <td>Binding affinity to immobilized human TFPI-1al...</td>\n",
       "      <td>115023</td>\n",
       "      <td>CHEMBL3713062</td>\n",
       "      <td>698</td>\n",
       "      <td>Tissue factor pathway inhibitor</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>1998110</td>\n",
       "      <td>CHEMBL4649949</td>\n",
       "      <td>Phenotypic growth assay for Mycobacterium tube...</td>\n",
       "      <td>22226</td>\n",
       "      <td>CHEMBL612545</td>\n",
       "      <td>688</td>\n",
       "      <td>Unchecked</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>1528801</td>\n",
       "      <td>CHEMBL3705476</td>\n",
       "      <td>Spectrophotometric 384 Well Assay: Malonyl CoA...</td>\n",
       "      <td>10608</td>\n",
       "      <td>CHEMBL4829</td>\n",
       "      <td>662</td>\n",
       "      <td>Acetyl-CoA carboxylase 2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>672664</td>\n",
       "      <td>CHEMBL1267251</td>\n",
       "      <td>Antiplasmodial activity against Plasmodium fal...</td>\n",
       "      <td>50425</td>\n",
       "      <td>CHEMBL364</td>\n",
       "      <td>654</td>\n",
       "      <td>Plasmodium falciparum</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>1640340</td>\n",
       "      <td>CHEMBL3887063</td>\n",
       "      <td>kinase Assay: The final concentration of DMSO ...</td>\n",
       "      <td>100126</td>\n",
       "      <td>CHEMBL5145</td>\n",
       "      <td>643</td>\n",
       "      <td>Serine/threonine-protein kinase B-raf</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>1528121</td>\n",
       "      <td>CHEMBL3706037</td>\n",
       "      <td>IMAP-FP Assay: Activated ERK2 activity was det...</td>\n",
       "      <td>11638</td>\n",
       "      <td>CHEMBL4040</td>\n",
       "      <td>641</td>\n",
       "      <td>MAP kinase ERK2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>1641471</td>\n",
       "      <td>CHEMBL3888194</td>\n",
       "      <td>Nav1.7 In Vitro PX Assay: HEK 293 cells stably...</td>\n",
       "      <td>18061</td>\n",
       "      <td>CHEMBL4296</td>\n",
       "      <td>641</td>\n",
       "      <td>Sodium channel protein type IX alpha subunit</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>1528048</td>\n",
       "      <td>CHEMBL3706327</td>\n",
       "      <td>Enzyme Assay: The Aurora assays described here...</td>\n",
       "      <td>20014</td>\n",
       "      <td>CHEMBL4722</td>\n",
       "      <td>630</td>\n",
       "      <td>Serine/threonine-protein kinase Aurora-A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>1637324</td>\n",
       "      <td>CHEMBL3880339</td>\n",
       "      <td>Inhibition of human FGFR3 using Btn-Flt3 as su...</td>\n",
       "      <td>11926</td>\n",
       "      <td>CHEMBL2742</td>\n",
       "      <td>620</td>\n",
       "      <td>Fibroblast growth factor receptor 3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>1641705</td>\n",
       "      <td>CHEMBL3888428</td>\n",
       "      <td>HTRF Biochemical Assay: The reactions employed...</td>\n",
       "      <td>10919</td>\n",
       "      <td>CHEMBL2835</td>\n",
       "      <td>620</td>\n",
       "      <td>Tyrosine-protein kinase JAK1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>1637322</td>\n",
       "      <td>CHEMBL3880337</td>\n",
       "      <td>Inhibition of human FGFR1 using Btn-Flt3 as su...</td>\n",
       "      <td>10266</td>\n",
       "      <td>CHEMBL3650</td>\n",
       "      <td>618</td>\n",
       "      <td>Fibroblast growth factor receptor 1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>1637323</td>\n",
       "      <td>CHEMBL3880338</td>\n",
       "      <td>Inhibition of human FGFR2 using Btn-Flt3 as su...</td>\n",
       "      <td>10264</td>\n",
       "      <td>CHEMBL4142</td>\n",
       "      <td>617</td>\n",
       "      <td>Fibroblast growth factor receptor 2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>1641706</td>\n",
       "      <td>CHEMBL3888429</td>\n",
       "      <td>HTRF Biochemical Assay: The reactions employed...</td>\n",
       "      <td>10938</td>\n",
       "      <td>CHEMBL2971</td>\n",
       "      <td>607</td>\n",
       "      <td>Tyrosine-protein kinase JAK2</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    assay_id assay_chembl_id  \\\n",
       "0    1641660   CHEMBL3888383   \n",
       "1    1528837   CHEMBL3705899   \n",
       "2    1527879   CHEMBL3705123   \n",
       "3    1528786   CHEMBL3706373   \n",
       "4    1641435   CHEMBL3888158   \n",
       "5    1528055   CHEMBL3705362   \n",
       "6    1642374   CHEMBL3889097   \n",
       "7    1638439   CHEMBL3881499   \n",
       "8    1998110   CHEMBL4649949   \n",
       "9    1528801   CHEMBL3705476   \n",
       "10    672664   CHEMBL1267251   \n",
       "11   1640340   CHEMBL3887063   \n",
       "12   1528121   CHEMBL3706037   \n",
       "13   1641471   CHEMBL3888194   \n",
       "14   1528048   CHEMBL3706327   \n",
       "15   1637324   CHEMBL3880339   \n",
       "16   1641705   CHEMBL3888428   \n",
       "17   1637322   CHEMBL3880337   \n",
       "18   1637323   CHEMBL3880338   \n",
       "19   1641706   CHEMBL3888429   \n",
       "\n",
       "                                          description     tid  \\\n",
       "0   Cell Based Assay: TRKA co-expressed with p75: ...   11902   \n",
       "1   In Vitro Assay: Recombinant human factor D (ex...  105346   \n",
       "2   Inhibiton Assay: Inhibition assay using human ...   11904   \n",
       "3   Homogeneous Time Resolved Fluorescence Assay: ...   10102   \n",
       "4   PARP Enzyme Inhibition Assay: PARP1 assay was ...   11663   \n",
       "5   Mobility-Shift kinase assay: Compounds of the ...   10906   \n",
       "6   Functional Assay: The functional activity of c...  103857   \n",
       "7   Binding affinity to immobilized human TFPI-1al...  115023   \n",
       "8   Phenotypic growth assay for Mycobacterium tube...   22226   \n",
       "9   Spectrophotometric 384 Well Assay: Malonyl CoA...   10608   \n",
       "10  Antiplasmodial activity against Plasmodium fal...   50425   \n",
       "11  kinase Assay: The final concentration of DMSO ...  100126   \n",
       "12  IMAP-FP Assay: Activated ERK2 activity was det...   11638   \n",
       "13  Nav1.7 In Vitro PX Assay: HEK 293 cells stably...   18061   \n",
       "14  Enzyme Assay: The Aurora assays described here...   20014   \n",
       "15  Inhibition of human FGFR3 using Btn-Flt3 as su...   11926   \n",
       "16  HTRF Biochemical Assay: The reactions employed...   10919   \n",
       "17  Inhibition of human FGFR1 using Btn-Flt3 as su...   10266   \n",
       "18  Inhibition of human FGFR2 using Btn-Flt3 as su...   10264   \n",
       "19  HTRF Biochemical Assay: The reactions employed...   10938   \n",
       "\n",
       "   target_chembl_id  cnt                                     pref_name  \n",
       "0        CHEMBL2815  794            Nerve growth factor receptor Trk-A  \n",
       "1     CHEMBL2176771  768                           Complement factor D  \n",
       "2        CHEMBL2998  749                            P2X purinoceptor 3  \n",
       "3        CHEMBL4550  733             5-lipoxygenase activating protein  \n",
       "4        CHEMBL3105  724                Poly [ADP-ribose] polymerase-1  \n",
       "5        CHEMBL2599  720                   Tyrosine-protein kinase SYK  \n",
       "6     CHEMBL1667665  700       Transient receptor potential M8 protein  \n",
       "7     CHEMBL3713062  698               Tissue factor pathway inhibitor  \n",
       "8      CHEMBL612545  688                                     Unchecked  \n",
       "9        CHEMBL4829  662                      Acetyl-CoA carboxylase 2  \n",
       "10        CHEMBL364  654                         Plasmodium falciparum  \n",
       "11       CHEMBL5145  643         Serine/threonine-protein kinase B-raf  \n",
       "12       CHEMBL4040  641                               MAP kinase ERK2  \n",
       "13       CHEMBL4296  641  Sodium channel protein type IX alpha subunit  \n",
       "14       CHEMBL4722  630      Serine/threonine-protein kinase Aurora-A  \n",
       "15       CHEMBL2742  620           Fibroblast growth factor receptor 3  \n",
       "16       CHEMBL2835  620                  Tyrosine-protein kinase JAK1  \n",
       "17       CHEMBL3650  618           Fibroblast growth factor receptor 1  \n",
       "18       CHEMBL4142  617           Fibroblast growth factor receptor 2  \n",
       "19       CHEMBL2971  607                  Tyrosine-protein kinase JAK2  "
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.head(20)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2022-11-28T13:04:19.673012Z",
     "start_time": "2022-11-28T13:04:19.670097Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "132"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "aids = df.assay_id.to_list()\n",
    "achemblids = df.assay_chembl_id.to_list()\n",
    "\n",
    "len(aids)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Generate the intake data catalog file "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2022-11-28T13:04:28.500229Z",
     "start_time": "2022-11-28T13:04:25.198458Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "794 rows affected.\n",
      "1 rows affected.\n",
      "766 rows affected.\n",
      "1 rows affected.\n",
      "748 rows affected.\n",
      "1 rows affected.\n",
      "733 rows affected.\n",
      "1 rows affected.\n",
      "724 rows affected.\n",
      "1 rows affected.\n",
      "720 rows affected.\n",
      "1 rows affected.\n",
      "700 rows affected.\n",
      "1 rows affected.\n",
      "698 rows affected.\n",
      "1 rows affected.\n",
      "688 rows affected.\n",
      "1 rows affected.\n",
      "662 rows affected.\n",
      "1 rows affected.\n",
      "653 rows affected.\n",
      "1 rows affected.\n",
      "643 rows affected.\n",
      "1 rows affected.\n",
      "641 rows affected.\n",
      "1 rows affected.\n",
      "641 rows affected.\n",
      "1 rows affected.\n",
      "630 rows affected.\n",
      "1 rows affected.\n",
      "620 rows affected.\n",
      "1 rows affected.\n",
      "620 rows affected.\n",
      "1 rows affected.\n",
      "618 rows affected.\n",
      "1 rows affected.\n",
      "617 rows affected.\n",
      "1 rows affected.\n",
      "607 rows affected.\n",
      "1 rows affected.\n",
      "601 rows affected.\n",
      "1 rows affected.\n",
      "600 rows affected.\n",
      "1 rows affected.\n",
      "597 rows affected.\n",
      "1 rows affected.\n",
      "596 rows affected.\n",
      "1 rows affected.\n",
      "587 rows affected.\n",
      "1 rows affected.\n",
      "587 rows affected.\n",
      "1 rows affected.\n",
      "584 rows affected.\n",
      "1 rows affected.\n",
      "582 rows affected.\n",
      "1 rows affected.\n",
      "574 rows affected.\n",
      "1 rows affected.\n",
      "574 rows affected.\n",
      "1 rows affected.\n",
      "574 rows affected.\n",
      "1 rows affected.\n",
      "571 rows affected.\n",
      "1 rows affected.\n",
      "566 rows affected.\n",
      "1 rows affected.\n",
      "565 rows affected.\n",
      "1 rows affected.\n",
      "564 rows affected.\n",
      "1 rows affected.\n",
      "561 rows affected.\n",
      "1 rows affected.\n",
      "554 rows affected.\n",
      "1 rows affected.\n",
      "550 rows affected.\n",
      "1 rows affected.\n",
      "545 rows affected.\n",
      "1 rows affected.\n",
      "533 rows affected.\n",
      "1 rows affected.\n",
      "511 rows affected.\n",
      "1 rows affected.\n",
      "504 rows affected.\n",
      "1 rows affected.\n",
      "498 rows affected.\n",
      "1 rows affected.\n",
      "495 rows affected.\n",
      "1 rows affected.\n",
      "483 rows affected.\n",
      "1 rows affected.\n",
      "480 rows affected.\n",
      "1 rows affected.\n",
      "478 rows affected.\n",
      "1 rows affected.\n",
      "468 rows affected.\n",
      "1 rows affected.\n",
      "465 rows affected.\n",
      "1 rows affected.\n",
      "465 rows affected.\n",
      "1 rows affected.\n",
      "464 rows affected.\n",
      "1 rows affected.\n",
      "460 rows affected.\n",
      "1 rows affected.\n",
      "458 rows affected.\n",
      "1 rows affected.\n",
      "457 rows affected.\n",
      "1 rows affected.\n",
      "448 rows affected.\n",
      "1 rows affected.\n",
      "446 rows affected.\n",
      "1 rows affected.\n",
      "445 rows affected.\n",
      "1 rows affected.\n",
      "442 rows affected.\n",
      "1 rows affected.\n",
      "440 rows affected.\n",
      "1 rows affected.\n",
      "433 rows affected.\n",
      "1 rows affected.\n",
      "432 rows affected.\n",
      "1 rows affected.\n",
      "430 rows affected.\n",
      "1 rows affected.\n",
      "428 rows affected.\n",
      "1 rows affected.\n",
      "427 rows affected.\n",
      "1 rows affected.\n",
      "426 rows affected.\n",
      "1 rows affected.\n",
      "425 rows affected.\n",
      "1 rows affected.\n",
      "419 rows affected.\n",
      "1 rows affected.\n",
      "419 rows affected.\n",
      "1 rows affected.\n",
      "408 rows affected.\n",
      "1 rows affected.\n",
      "404 rows affected.\n",
      "1 rows affected.\n",
      "403 rows affected.\n",
      "1 rows affected.\n",
      "402 rows affected.\n",
      "1 rows affected.\n",
      "401 rows affected.\n",
      "1 rows affected.\n",
      "398 rows affected.\n",
      "1 rows affected.\n",
      "398 rows affected.\n",
      "1 rows affected.\n",
      "395 rows affected.\n",
      "1 rows affected.\n",
      "387 rows affected.\n",
      "1 rows affected.\n",
      "383 rows affected.\n",
      "1 rows affected.\n",
      "383 rows affected.\n",
      "1 rows affected.\n",
      "382 rows affected.\n",
      "1 rows affected.\n",
      "381 rows affected.\n",
      "1 rows affected.\n",
      "377 rows affected.\n",
      "1 rows affected.\n",
      "377 rows affected.\n",
      "1 rows affected.\n",
      "376 rows affected.\n",
      "1 rows affected.\n",
      "370 rows affected.\n",
      "1 rows affected.\n",
      "368 rows affected.\n",
      "1 rows affected.\n",
      "367 rows affected.\n",
      "1 rows affected.\n",
      "367 rows affected.\n",
      "1 rows affected.\n",
      "365 rows affected.\n",
      "1 rows affected.\n",
      "365 rows affected.\n",
      "1 rows affected.\n",
      "364 rows affected.\n",
      "1 rows affected.\n",
      "362 rows affected.\n",
      "1 rows affected.\n",
      "363 rows affected.\n",
      "1 rows affected.\n",
      "362 rows affected.\n",
      "1 rows affected.\n",
      "361 rows affected.\n",
      "1 rows affected.\n",
      "357 rows affected.\n",
      "1 rows affected.\n",
      "357 rows affected.\n",
      "1 rows affected.\n",
      "356 rows affected.\n",
      "1 rows affected.\n",
      "352 rows affected.\n",
      "1 rows affected.\n",
      "351 rows affected.\n",
      "1 rows affected.\n",
      "349 rows affected.\n",
      "1 rows affected.\n",
      "347 rows affected.\n",
      "1 rows affected.\n",
      "347 rows affected.\n",
      "1 rows affected.\n",
      "346 rows affected.\n",
      "1 rows affected.\n",
      "342 rows affected.\n",
      "1 rows affected.\n",
      "341 rows affected.\n",
      "1 rows affected.\n",
      "339 rows affected.\n",
      "1 rows affected.\n",
      "334 rows affected.\n",
      "1 rows affected.\n",
      "333 rows affected.\n",
      "1 rows affected.\n",
      "330 rows affected.\n",
      "1 rows affected.\n",
      "329 rows affected.\n",
      "1 rows affected.\n",
      "325 rows affected.\n",
      "1 rows affected.\n",
      "325 rows affected.\n",
      "1 rows affected.\n",
      "324 rows affected.\n",
      "1 rows affected.\n",
      "318 rows affected.\n",
      "1 rows affected.\n",
      "318 rows affected.\n",
      "1 rows affected.\n",
      "317 rows affected.\n",
      "1 rows affected.\n",
      "317 rows affected.\n",
      "1 rows affected.\n",
      "317 rows affected.\n",
      "1 rows affected.\n",
      "316 rows affected.\n",
      "1 rows affected.\n",
      "313 rows affected.\n",
      "1 rows affected.\n",
      "310 rows affected.\n",
      "1 rows affected.\n",
      "300 rows affected.\n",
      "1 rows affected.\n",
      "306 rows affected.\n",
      "1 rows affected.\n",
      "306 rows affected.\n",
      "1 rows affected.\n",
      "306 rows affected.\n",
      "1 rows affected.\n",
      "305 rows affected.\n",
      "1 rows affected.\n",
      "304 rows affected.\n",
      "1 rows affected.\n",
      "303 rows affected.\n",
      "1 rows affected.\n",
      "301 rows affected.\n",
      "1 rows affected.\n",
      "300 rows affected.\n",
      "1 rows affected.\n",
      "300 rows affected.\n",
      "1 rows affected.\n",
      "  CHEMBL3888383:\n",
      "    description: \"Assay CHEMBL3888383: Cell Based Assay: TRKA co-expressed with p75: The assays are based upon DiscoveRx's proprietary Enzyme Fragment Complementation (EFC) technology. In the case of the TRK cell lines, the enzyme acceptor (EA) protein is fused to a SH2 protein and the TRK receptor of interest has been tagged with a Prolink tag. Cryopreserved PathHunter cells are used from either in-house produced batches or bulk batches bought directly from DiscoveRx. Cryopreserved cells are resuscitated, spun 1000 rpm for 4 min to remove freezing media, and resuspended in MEM+0.5% horse serum (both Invitrogen) to 5e^5 cells/ml. The cells are then plated using a Multidrop into Greiner white tissue culture treated plates at 20 μl/well and incubated for 24 h at 37 °C., 5% CO2, high humidity. On the day of the assay, the cell plates are allowed to cool to room temperature for 30 min prior to the assay. 4 mM stock solutions of test compounds are prepared and serially diluted in 100% DMSO. A standard curve using the compound of Example 135, WO2005/116035 at a top concentration of 150 μM is also prepared on each test plate. High percentage effect (HPE) is defined by 150 μM of the compound of Example 135, WO2005/116035 and 0% effect (ZPE) is defined by 100% DMSO. Plates containing 1 μl of serially diluted compound, standard and HPE/ZPE are diluted 1/66 in assay buffer (PBS minus Ca2+, minus Mg2+ with 0.05% pluronic F127) using a Wellmate. Using a Platemate Plus, 5 μl of 1/66 diluted test compounds is then transferred to the cell plate and allowed to reach equilibrium by incubating for 30 min at room temperature before addition of agonist stimulus: 10 μl/well of 2 nM (0.571 nM FAC) of the cognate neurotrophin (Peprotech) diluted in agonist buffer (HBSS with 0.25% BSA). Final assay concentration of the test compounds is 8.66 μM, (the compound of Example 135, WO2005/116035 FAC is 0.325 μM). The plates are left at room temperature for a further 2 hours before addition of 10 μl of the DiscoveRx PathHunter detection reagent (made up by adding 1 part Galacton Star, 5 parts Emerald II and 19 parts Cell Assay Buffer as per the manufacturer's instructions). After reagent addition, plates are covered and incubated at room temperature for 60 minutes. Luminescence signal is read using an Envision.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3888383.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3888383/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.30\n",
      "      max pchembl_value: 8.99\n",
      "      median pchembl_value: 7.46\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705899:\n",
      "    description: \"Assay CHEMBL3705899: In Vitro Assay: Recombinant human factor D (expressed in E. coli and purified using standard methods) at 10 nM concentration is incubated with test compound at various concentrations for 1 hour at room temperature in 0.1 M Hepes buffer, pH 7.5, containing 1 mM MgCl2, 1 M NaCl and 0.05% CHAPS. A synthetic substrate Z-Lys-thiobenzyl and 2,4-dinitrobenzenesulfonyl-fluoresceine are added to final concentrations of 200 μM and 25 μM, respectively. The increase in fluorescence is recorded at excitation of 485 nm and emission at 535 nm in a microplate spectrofluorimeter. IC50 values are calculated from percentage of inhibition of complement factor D-activity as a function of test compound concentration.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705899.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705899/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.61\n",
      "      max pchembl_value: 9.00\n",
      "      median pchembl_value: 6.96\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705123:\n",
      "    description: \"Assay CHEMBL3705123: Inhibiton Assay: Inhibition assay using human P2X3 receptor gene (GeneBank accession number Y07683).\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705123.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705123/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 6.00\n",
      "      max pchembl_value: 8.70\n",
      "      median pchembl_value: 7.09\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3706373:\n",
      "    description: \"Assay CHEMBL3706373: Homogeneous Time Resolved Fluorescence Assay: The assay below is used to test the modulatory activity of compounds against FLAP. Human and mouse FLAP-encoding DNA was amplified by polymerase chain reaction and cloned into pFastBacl (Invitrogen) with a NH2-terminal 6-His tag for expression in Spodoptera frugiperda (Sf-9) cells. FLAP-containing membranes were prepared as was a FITC-labeled FLAP modulator [5-[({[2-(2-{3-[3-(tert-Butylsulfanyl)-1-(4-chlorobenzyl)-5-(quinolin-2-ylmethoxy)-1H-indol-2-yl]-2,2-dimethylpropanoyl}hydrazino)-2-oxoethyl]sulfanyl}acetyl)amino]-2-(6-hydroxy-3-oxo-3H-xanthen-9-yl)benzoic acid]. The FLAP binding assay is performed in HTRF format (homogeneous time resolved fluorescence). FLAP-containing membranes (1 ug/well final for human) are incubated in the presence of the HTRF ligand, [5-[({[2-(2-{3-[3-(tert-butylsulfanyl)-1-(4-chlorobenzyl)-5-(quinolin-2-ylmethoxy)-1H-indol-2-yl]-2,2-dimethylpropanoyl}hydrazino)-2-oxoethyl]sulfanyl}acetyl)amino]-2-(6-hydroxy-3-oxo-3H-xanthen-9-yl)benzoic acid].\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3706373.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3706373/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: Ki\n",
      "      min pchembl_value: 5.60\n",
      "      max pchembl_value: 9.40\n",
      "      median pchembl_value: 7.70\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3888158:\n",
      "    description: \"Assay CHEMBL3888158: PARP Enzyme Inhibition Assay: PARP1 assay was conducted in PARP assay buffer containing 50 mM Tris pH 8.0, 1 mM DTT, 4 mM MgCl2. PARP reactions contained 1.5 uM [3H]-NAD+ (1.6 uCi/mmol), 200 nM biotinylated histone H1, 200 nM sIDNA, and 1 nM PARP enzyme. Auto reactions utilizing SPA bead-based detection were carried out in 100 uL volumes in white 96 well plates. Reactions were initiated by adding 50 ul of 2xNAD+ substrate mixture to 50 uL of 2x enzyme mixture containing PARP and DNA. These reactions were terminated by the addition of 150 uL of 1.5 mM benzamide (~1000-fold over its IC50). 170 uL of the stopped reaction mixtures were transferred to streptavidin Flash Plates, incubated for 1 hour, and counted using a TopCount microplate scintillation counter.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3888158.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3888158/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: Ki\n",
      "      min pchembl_value: 5.04\n",
      "      max pchembl_value: 10.00\n",
      "      median pchembl_value: 7.52\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705362:\n",
      "    description: \"Assay CHEMBL3705362: Mobility-Shift kinase assay: Compounds of the examples provided herein were assessed for their ability to inhibit Syk kinase by utilizing Caliper Life Sciences' proprietary LabChip technology. The off-chip incubation mobility-shift kinase assay uses a microfluidic chip to measure the conversion of a fluorescent peptide substrate to a phosphorylated product. The reaction mixture, from a microtiter plate well, is introduced through a capillary sipper onto the chip, where the nonphosphorylated substrate and phosphorylated product are separated by electrophoresis and detected via laser induced fluorescence. The signature of the fluorescence signal over time reveals the extent of the reaction. The phosphorylated product migrates through the chip faster than the non-phosphorylated substrate, and signals from the two forms of the peptide appear as distinct peaks.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705362.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705362/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.68\n",
      "      max pchembl_value: 8.41\n",
      "      median pchembl_value: 6.63\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3889097:\n",
      "    description: \"Assay CHEMBL3889097: Functional Assay: The functional activity of compounds of the formula (I) was determined by measuring changes in intracellular calcium concentration using a Ca2+-sensitive fluorescent dye. The changes in fluorescent signal were monitored by a fluorescence plate reader, either a FLIPR (Molecular Devices) or FDSS (Hamamatsu). Increases in intracellular Ca2+ concentration were readily detected upon activation with icilin.At 24 hrs prior to assay, HEK293 cells stably expressing canine TRPM8 were seeded in culture medium in black wall, clear-base poly-D-lysine coated 384-well plates (BD Biosciences, NJ, USA) and grown overnight in 5% CO2 at 37° C. On assay day, growth media was removed and cells were loaded with Calcium 3 Dye (Molecular Devices) for 35 min at 37° C., under 5% CO2 and then for 25 min at room temperature and atmosphere. Subsequently, cells were tested for agonist-induced increases in intracellular Ca2+ levels using FLIPR or FDSS. Cells were challenged with a compound of the Formula (I) (at varying concentrations) and intracellular Ca2+ was measured for 5 min prior to the addition of icilin to all wells to achieve a final concentration that produces approximately an 80% maximal response. EC50 or IC50 values for compounds of the present invention were determined from eight-point dose-response studies.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3889097.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3889097/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 6.46\n",
      "      max pchembl_value: 9.40\n",
      "      median pchembl_value: 7.77\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3881499:\n",
      "    description: \"Assay CHEMBL3881499: Binding affinity to immobilized human TFPI-1alpha assessed as inhibition of biotinylated tracer peptide JBT-0131 binding incubated for 1.5 hrs by measuring horseradish peroxidase-conjugated streptavidin-mediated conversion of TMB substrate by screening ELISA based competition assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3881499.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3881499/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.21\n",
      "      max pchembl_value: 8.05\n",
      "      median pchembl_value: 6.69\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL4649949:\n",
      "    description: \"Assay CHEMBL4649949: Phenotypic growth assay for Mycobacterium tuberculosis grown for 3 days on 7H9, glucose tyloxapol based media\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL4649949.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL4649949/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.00\n",
      "      max pchembl_value: 6.70\n",
      "      median pchembl_value: 4.53\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705476:\n",
      "    description: \"Assay CHEMBL3705476: Spectrophotometric 384 Well Assay: Malonyl CoA formation by acetyl CoA carboxylases is stoichometrically linked to the consumption of ATP. ACC2 activity is measured in a NADH-linked kinetic method measuring ADP generated during the ACC reaction using a coupled lactate dehydrogenase/pyruvate kinase reaction.For biological testing, a human ACC2 construct which lacks the 128 amino acids at the N-terminus for increased solubility (nt 385-6966 in Genbank entry AJ575592) is cloned. The protein is then expressed in insect cells using a baculoviral expression system. Protein purification is performed by anion exchange.All compounds are dissolved in dimethyl sulfoxide (DMSO) to a concentration of 10 mM.Assay reactions are then carried out in 384-well plates, with hACC2 in an appropriate dilution and at final assay concentrations (f.c.) of 100 mM Tris (pH 7.5), 10 mM trisodium citrate, 25 mM KHCO3, 10 mM MgCl2, 0.5 mg/mL BSA, 3.75 mM reduced L-glutathione, 15 U/mL lactate dehydrogenase, 0.5 mM phosphoenolpyruvate.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705476.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705476/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.52\n",
      "      max pchembl_value: 7.30\n",
      "      median pchembl_value: 6.13\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL1267251:\n",
      "    description: \"Assay CHEMBL1267251: Antiplasmodial activity against Plasmodium falciparum W2 after 72 hrs by SYBR green assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL1267251.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL1267251/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.80\n",
      "      max pchembl_value: 10.15\n",
      "      median pchembl_value: 5.20\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3887063:\n",
      "    description: \"Assay CHEMBL3887063: kinase Assay: The final concentration of DMSO is 5%. 10 μL of the B-Raf (V600E)-kinase solution are pipetted in (containing 0.5 ng B-Raf (V600E)-kinase in 20 mM Tris-HCl pH 7.5, 0.1 mM EDTA, 0.1 mM EGTA, 0.286 mM sodium orthovanadate, 10% glycerol, 1 mg/mL bovine serum albumin, 1 mM dithiothreitol) and the mixture is incubated for 24 h at RT under with shaking. The kinase reaction is started by the addition of 20 μL ATP solution [final concentration: 250 μM ATP, 30 mM Tris-HCl pH 7.5, 0.02% Brij, 0.2 mM sodium orthovanadate, 10 mM magnesium acetate, 0.1 mM EGTA, phosphatase cocktail (Sigma, # P2850, dilution recommended by the manufacturer), 0.1 mM EGTA] and 10 μL MEK1 solution [containing 50 ng biotinylated MEK1 (prepared from purified MEK1 according to standard procedure, e.g. with EZ-Link Sulpho-NHS-LC-Biotin reagent, Pierce, #21335) and carried out for 60 min at RT with constant shaking.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3887063.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3887063/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.82\n",
      "      max pchembl_value: 9.00\n",
      "      median pchembl_value: 8.40\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3706037:\n",
      "    description: \"Assay CHEMBL3706037: IMAP-FP Assay: Activated ERK2 activity was determined in an IMAP-FP assay (Molecular Devices). Using this assay format, the potency (IC50) of each compound was determined from a 10 point (1:3 serial dilution, 3 uM starting compound concentration) titration curve using the following outlined procedure. To each well of a black Corning 384-well plate (Corning Catalog #3575), 7.5 mL of compound (3333 fold dilution in final assay volume of 25 uL) was dispensed, followed by the addition of 15 uL of kinase buffer (tween containing kinase buffer, Molecular Devices) containing 0.0364 ng/mL (0.833 nM) of phosphorylated active hERK2 enzyme. Following a 15 minute compound & enzyme incubation, each reaction was initiated by the addition of 10 uL kinase buffer containing 2.45 uM ERK2 IMAP substrate peptides (2.25 uM-unlabeled IPTTPITTTYFFFK-COOH and 200 nM-labeled IPTTPITTTYFFFK-5FAM (5-carboxyfluorescein)-COOH), and 75 uM ATP.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3706037.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3706037/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.61\n",
      "      max pchembl_value: 9.82\n",
      "      median pchembl_value: 8.49\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3888194:\n",
      "    description: \"Assay CHEMBL3888194: Nav1.7 In Vitro PX Assay: HEK 293 cells stably transfected with human Nav1.7 were recorded in whole cell voltage clamp mode with the PatchXpress automated electrophysiology system (Molecular Devices, LLC, Sunnyvale, Calif.). Compound effects were measured on a partially inactivated state of the sodium channel. Cells were clamped to a holding potential yielding 20 to 50% inactivation. To elicit sodium current, channels were activated by pulsing to -10 mV for 20 msec. This voltage protocol was repeated at a rate of 0.1 Hz throughout the experiment. A single concentration of test compound was applied to cells for a duration of 3 minutes.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3888194.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3888194/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.49\n",
      "      max pchembl_value: 8.52\n",
      "      median pchembl_value: 7.25\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3706327:\n",
      "    description: \"Assay CHEMBL3706327: Enzyme Assay: The Aurora assays described here are performed on two Caliper Life Sciences systems: the LC3000 and the Desktop Profiler. These provide data on enzyme activity via measurement of the relative amounts of phosphorylated or unphosphorylated fluorescently labelled substrate peptide at the end of an enzymatic reaction. These different states of peptide are resolved by applying a potential difference across the sample. The presence of the charged phosphate group on the product (as opposed to the substrate) causes a different peptide mobility between the two peptides. This is visualized by excitation of the fluorescent label on the substrate and product peptides and represented as peaks within the analysis software.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3706327.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3706327/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.00\n",
      "      max pchembl_value: 11.00\n",
      "      median pchembl_value: 8.38\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3880339:\n",
      "    description: \"Assay CHEMBL3880339: Inhibition of human FGFR3 using Btn-Flt3 as substrate after 60 mins by TR-FRET assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3880339.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3880339/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.40\n",
      "      max pchembl_value: 9.35\n",
      "      median pchembl_value: 8.34\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3888428:\n",
      "    description: \"Assay CHEMBL3888428: HTRF Biochemical Assay: The reactions employed a common peptide substrate, LCB-EQEDEPEGDYFEWLW-NH2 (in-house). The basic assay protocol is as follows: First, 250 nL of diluted compounds in DMSO were dispensed into the wells of a dry 384-well Black plate (Greiner #781076) using a Labcyte Echo 555 acoustic dispenser. Subsequent reagent additions employed an Agilent Bravo. Next, 18 uL of 1.11x enzyme and 1.11x substrate in 1x assay buffer (Invitrogen kinase buffer # PV3189, 2 mM DTT, 0.05% BSA) were added to the wells and shaken and then preincubated for 30 minutes at room temperature to allow compound binding to equilibrate. After equilibration, 2 uL of 10xATP in 1x assay buffer was added to initiate the kinase reaction and the plates were shaken and then incubated at room temperature for 120 minutes. At the end of the incubation, 20 uL of 2x stop buffer (streptavidin-Dylight 650 (Thermo #84547B/100 mL), Eu-tagged pY20 antibody (Perkin Elmer #AD0067), EDTA, HEPES, and Triton) was added to quench the reaction.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3888428.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3888428/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 6.19\n",
      "      max pchembl_value: 11.00\n",
      "      median pchembl_value: 8.40\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3880337:\n",
      "    description: \"Assay CHEMBL3880337: Inhibition of human FGFR1 using Btn-Flt3 as substrate after 60 mins by TR-FRET assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3880337.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3880337/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.10\n",
      "      max pchembl_value: 9.44\n",
      "      median pchembl_value: 8.33\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3880338:\n",
      "    description: \"Assay CHEMBL3880338: Inhibition of human FGFR2 using Btn-Flt3 as substrate after 60 mins by TR-FRET assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3880338.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3880338/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.20\n",
      "      max pchembl_value: 9.05\n",
      "      median pchembl_value: 8.21\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3888429:\n",
      "    description: \"Assay CHEMBL3888429: HTRF Biochemical Assay: The reactions employed a common peptide substrate, LCB-EQEDEPEGDYFEWLW-NH2 (in-house). The basic assay protocol is as follows: First, 250 nL of diluted compounds in DMSO were dispensed into the wells of a dry 384-well Black plate (Greiner #781076) using a Labcyte Echo 555 acoustic dispenser. Subsequent reagent additions employed an Agilent Bravo. Next, 18 uL of 1.11x enzyme and 1.11x substrate in 1x assay buffer (Invitrogen kinase buffer # PV3189, 2 mM DTT, 0.05% BSA) were added to the wells and shaken and then preincubated for 30 minutes at room temperature to allow compound binding to equilibrate. After equilibration, 2 uL of 10xATP in 1x assay buffer was added to initiate the kinase reaction and the plates were shaken and then incubated at room temperature for 120 minutes. At the end of the incubation, 20 uL of 2x stop buffer (streptavidin-Dylight 650 (Thermo #84547B/100 mL), Eu-tagged pY20 antibody (Perkin Elmer #AD0067), EDTA, HEPES, and Triton) was added to quench the reaction.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3888429.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3888429/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.83\n",
      "      max pchembl_value: 10.00\n",
      "      median pchembl_value: 7.15\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3880344:\n",
      "    description: \"Assay CHEMBL3880344: Inhibition of FGFR3 (unknown origin) transfected in mouse BA/F3 cells assessed as decrease in cell proliferation in absence of mouse IL3 after 24 hrs by Alamar blue assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3880344.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3880344/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.00\n",
      "      max pchembl_value: 9.42\n",
      "      median pchembl_value: 6.77\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705873:\n",
      "    description: \"Assay CHEMBL3705873: Kinase Assay: MELK activity was determined in the presence or absence of compounds using fluorescein isothiocyanate-labeled (FITC-labeled) histone H3 peptide as a substrate. The extent of FITC-labeled histone H3 peptide phosphorylation was measured by immobilized metal ion affinity-based fluorescence polarization (IMAP) technology (Sportsman J R, et al., Assay Drug Dev. Technol. 2: 205-14, 2004) using IMAP FP Progressive Binding System (Molecular Devices Corporation). Test compounds were dissolved in DMSO at 12.5 mM and then serially diluted as the DMSO concentration in the assays to be 1%. The serially diluted compounds, 0.8 ng/micro-L PBK (Carna Biosciences) and 100 nM FITC-labeled histone H3 peptide were reacted in a reaction buffer (20 mM HEPES, 0.01% Tween-20, 0.3 mM MgCl2, 2 mM dithiothreitol, 50 micro-M ATP, pH 7.4) at room temperature for 1 hour. The reaction was stopped by the addition of three fold assay volume of progressive binding solution. Following 0.5 hour incubation at room temperature.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705873.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705873/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.29\n",
      "      max pchembl_value: 9.18\n",
      "      median pchembl_value: 8.27\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3880342:\n",
      "    description: \"Assay CHEMBL3880342: Inhibition of FGFR1 (unknown origin) transfected in mouse BA/F3 cells assessed as decrease in cell proliferation in absence of mouse IL3 after 24 hrs by Alamar blue assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3880342.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3880342/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.02\n",
      "      max pchembl_value: 9.22\n",
      "      median pchembl_value: 6.64\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705869:\n",
      "    description: \"Assay CHEMBL3705869: Inhibition Assay: The inhibitory action of PDHK activity was indirectly evaluated by performing a kinase reaction in the presence of a test compound and measuring the residual PDH activity. For expression of hPDHK2 activity, Escherichia coli strain BL21(DE3) cells (Novagen) were transformed with the pET17b vector containing modified hPDHK2 cDNA. Escherichia coli were grown to an optical density 0.6 (600 nmol/L) at 30C. Protein expression was induced by the addition of 500 umol/L isopropyl-beta-thiogalactopyranoside. Escherichia coli were cultured at 30 C. for 5 hr and harvested by centrifugation. Resuspension of the Escherichia coli paste was disrupted by a microfluidizer. FLAG-Tagged protein was separated using FLAG affinity gel (Sigma). The gel was washed with 20 mmol/L N-(2-hydroxyethyl)piperazine-N'-2-ethanesulfonic acid-sodium hydroxide (HEPES-NaOH), 500 mmol/L sodium chloride, 1% ethylene glycol, and 0.1% Pluronic F-68 (pH 8.0), and the binding protein was eluted with 20 mmol/L HEPES.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705869.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705869/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.16\n",
      "      max pchembl_value: 7.85\n",
      "      median pchembl_value: 6.52\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3880340:\n",
      "    description: \"Assay CHEMBL3880340: Inhibition of human FGFR4 using Btn-Flt3 as substrate after 60 mins by TR-FRET assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3880340.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3880340/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.10\n",
      "      max pchembl_value: 8.91\n",
      "      median pchembl_value: 7.66\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL4649948:\n",
      "    description: \"Assay CHEMBL4649948: Phenotypic growth assay for Mycobacterium tuberculosis grown for 4 days on DPPC, cholesterol, tyloxapol based media\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL4649948.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL4649948/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.00\n",
      "      max pchembl_value: 6.74\n",
      "      median pchembl_value: 4.58\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705357:\n",
      "    description: \"Assay CHEMBL3705357: Enzyme Assay: P70S6K inhibitor compounds are diluted and plated in 96 well plates. A reaction mixture including the following components is then added to the compound plate to initiate the enzyme reaction; P70S6K (3 nM, T412E mutant, Millipore) is mixed with 24 μM ATP in an assay buffer containing 100 mM Hepes (pH 7.5), 5 mM MgCl2, 1 mM DTT, 0.015% Brij and 1 μM of the substrate peptide FITC-AHA-AKRRRLSSLRA-OH (derived from the S6 ribosomal protein sequence, FITC=fluorescein isothiocyanate, AHA=6-aminohexanoic acid). The reaction is incubated for 90 min at 25° C., before the addition of 10 mM EDTA to stop the reaction. The proportion of substrate and product (phosphorylated) peptide is analysed on a Caliper Life Sciences Lab Chip 3000, using a pressure of −1.4 psi, and upstream and downstream voltages of −3000 and −700 respectively.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705357.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705357/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.44\n",
      "      max pchembl_value: 9.92\n",
      "      median pchembl_value: 7.34\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3887118:\n",
      "    description: \"Assay CHEMBL3887118: Inhibition Assay: The assays were performed in U-bottom 384-well plates. The final assay volume was 30 L prepared from 15 L additions of enzyme and substrates (fluoresceinated peptide and ATP) and test compounds in assay buffer (20 mM HEPES pH 7.2, 10 mM MgCl2, 0.015% Brij 35 and 4 mM DTT). The reaction was initiated by the combination of IRAK4 with substrates and test compounds. The reaction mixture was incubated at room temperature for 60 min. and terminated by adding 45 μL of 35 mM EDTA to each sample. The reaction mixture was analyzed on the Caliper LABCHIP® 3000 (Caliper, Hopkinton, Mass.) by electrophoretic separation of the fluorescent substrate and phosphorylated product. Inhibition data were calculated by comparison to no enzyme control reactions for 100% inhibition and vehicle-only reactions for 0% inhibition. The final concentrations of reagents in the assays are ATP, 500 μM; FL-IPTSPITTTYFFFKKK peptide 1.5 μM; IRAK4, 0.6 nM; and DMSO, 1.6%.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3887118.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3887118/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 7.63\n",
      "      max pchembl_value: 9.05\n",
      "      median pchembl_value: 8.36\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3706088:\n",
      "    description: \"Assay CHEMBL3706088: Filter-Binding-Assay: PI3Kalpha assay described herein provides IC50 values indicating the activity of the compounds inhibiting PI3 kinase alpha activity Inhibition of PI3 kinase is expected to be indicative of activity in treating conditions of excessive or anomalous cell proliferation, such as cancers. See also J. A. Engelman, Nature Reviews Cancer, 2009, 9, 550-562; A. Carnero, Expert Opin. Investig. Drugs, 2009, 18, 1265-1277 and P. Liu et al., Nature Reviews Drug Discovery, 2009, 8, 627-64.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3706088.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3706088/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.51\n",
      "      max pchembl_value: 8.32\n",
      "      median pchembl_value: 6.81\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3706089:\n",
      "    description: \"Assay CHEMBL3706089: mTOR Kinase Activity Assay: The mTOR kinase TR-FRET assay utilizes a physiologically relevant protein substrate for mTOR (4E-BP1, labeled with an acceptor fluorophore (Green Fluorescent Protein) and paired with a corresponding Tb-labeled phospho-specific antibody. The assay itself can be divided into two phases: the reaction phase and the detection phase.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3706089.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3706089/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 6.56\n",
      "      max pchembl_value: 10.15\n",
      "      median pchembl_value: 8.30\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3706045:\n",
      "    description: \"Assay CHEMBL3706045: Inhibition Assay: The ability of the nuclear factor-kappa B (NF-kB)-inducing kinase (NIK) to catalyze the hydrolysis of adenosine-5'-triphosphate (ATP) was monitored using the Transcreener ADP (adenosine-5'-diphosphate) assay (BellBrook Labs). Purified NIK (0.2-1 nM) derived from a baculovirus-infected insect cell expression system was incubated with test compounds for 1-3.5 hours in 50 mM 2-[4-(2-hydroxyethyl)piperazin-1-yl]ethanesulfonic acid buffer (pH 7.2) containing 10 mM MgCl.sub.2, 2 mM dithiothreitol, 10 uM ATP, 0.01% Triton X-100, 0.1% gamma-globulins from bovine blood, 1% dimethylsulfoxide (DMSO), 12 ug/mL ADP antibody and 4 nM ADP-AlexaFluor.RTM. 633 tracer. Reactions were quenched by the addition of 20 mM 2,2',2'',2'''-(ethane-1,2-diyldinitrilo)tetraacetic acid and 0.01% Brij 35. The tracer bound to the antibody was displaced by the ADP generated during the NIK reaction, which causes a decrease in fluorescence polarization that was measured by laser excitation.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3706045.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3706045/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: Ki\n",
      "      min pchembl_value: 5.31\n",
      "      max pchembl_value: 10.40\n",
      "      median pchembl_value: 8.16\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL1267246:\n",
      "    description: \"Assay CHEMBL1267246: Antiplasmodial activity against Plasmodium falciparum 7G8 after 72 hrs by SYBR green assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL1267246.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL1267246/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.44\n",
      "      max pchembl_value: 10.22\n",
      "      median pchembl_value: 5.10\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL1267249:\n",
      "    description: \"Assay CHEMBL1267249: Antiplasmodial activity against Plasmodium falciparum GB4 after 72 hrs by SYBR green assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL1267249.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL1267249/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.70\n",
      "      max pchembl_value: 10.52\n",
      "      median pchembl_value: 5.00\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3889212:\n",
      "    description: \"Assay CHEMBL3889212: Scintillation Proximity Assay (SPA): The binding of potential ligands to RORγ is measured by competition with [3H] 25-hydroxycholesterol (Perkin Elmer NET674250UC) using a scintillation proximity assay (SPA) binding assay. The ligand binding domain of human RORγ (A262-S507) with an N-terminal His tag is expressed in E. coli and purified using nickel affinity chromatography. 15 ug/well RORγ (A262-S507) is incubated with test compound at varying concentrations in 3-fold serial dilution, with final concentrations ranging from 16.6 uM to 0.28 nM for 10 min at room temperature in PBS buffer (Invitrogen #14190-144) containing 0.5% fatty acid free BSA (Gemini Bio-Products, Cat. #700-107P) and 0.1% Glycerol (Sigma Cat#G5516). 10 nM of [3H] 25-hydroxycholesterol is then added, and the reaction is incubated for 10 min. 10 mg/mL of Copper-His Tag-PVT beads (Perkin Elmer cat #RPNQ0095) are added, and the mixture is incubated for 60 min. The reaction is read on a TopCount Microplate scintillation plate reader (Perkin Elmer). The competition data of the test compound over a range of concentrations was plotted as percentage inhibition of radioligand specifically bound in the absence of test compound (percent of total signal). After correcting for non-specific binding, IC50 values were determined. The IC50 value is defined as the concentration of test compound needed to reduce [3H] 25-hydroxycholesterol specific binding by 50% and is calculated using the four parameter logistic equation to fit the normalized data.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3889212.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3889212/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.30\n",
      "      max pchembl_value: 7.92\n",
      "      median pchembl_value: 6.89\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3888190:\n",
      "    description: \"Assay CHEMBL3888190: Time-Resolved Fluorescence Resonance Energy Transfer (TR-FRET) Binding Assay : Probe 1:The assay was carried out in 18 uL low volume plates (Owens Corning) in reaction buffer (50 mM HEPES (NaOH), pH 7.5, 100 mM NaCl, 10 mM MgCl2, 1 mM DTT, 1% Glycerol) using 6.8 nM recombinant, human, C-terminally-His tagged NAMPT, 1 nM Tb-anti-His antibody (Invitrogen, Cat #PV5895), and 200 nM probe (Oregon Green 488-conjugated APO0866; A-1251667.0. Plates were covered, and reactions were carried out for 2-3 hours. Plates were read with Envision (Laser Lantha low volume protocol) after 2 to 3 hours. Excitation was carried out at 337 nm, and the ratio of emission of Oregon Green (520 nm) to terbium (492 nm) was determined and used to calculate IC50 values of test compounds.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3888190.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3888190/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.90\n",
      "      max pchembl_value: 9.02\n",
      "      median pchembl_value: 7.27\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3706329:\n",
      "    description: \"Assay CHEMBL3706329: HTRF Assay: The ability of compounds to inhibit the activity of JAK1, JAK2, JAK3, and Tyk2 was measured using a recombinant purified GST-tagged catalytic domain for each enzyme (Invitrogen JAK1 #M4290, JAK2 #M4290, JAK3 #M4290, Tyk2 #M4290) in an HTRF format biochemical assay. The reactions employed a common peptide substrate, LCB-EQEDEPEGDYFEWLW-NH2 (in-house). The basic assay protocol is as follows: First, 250 mL of diluted compounds in DMSO were dispensed into the wells of a dry 384-well Black plate (Greiner #781076) using a Labcyte Echo 555 acoustic dispenser. Subsequent reagent additions employed an Agilent Bravo. Next, 18 uL of 1.11x enzyme and 1.11x substrate in 1x assay buffer (Invitrogen kinase buffer #PV3189, 2 mM DTT, 0.05% BSA) were added to the wells and shaken and then preincubated for 30 minutes at ambient temperature to allow compound binding to equilibrate. After equilibration, 2 uL of 10xATP in 1x assay buffer was added to initiate the kinase reaction.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3706329.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3706329/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 6.16\n",
      "      max pchembl_value: 10.40\n",
      "      median pchembl_value: 8.70\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3880341:\n",
      "    description: \"Assay CHEMBL3880341: Inhibition of human VEGFR2 using Btn-Flt3 as substrate after 120 mins by TR-FRET assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3880341.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3880341/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.30\n",
      "      max pchembl_value: 8.08\n",
      "      median pchembl_value: 7.02\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3889155:\n",
      "    description: \"Assay CHEMBL3889155: FRET-Based Membrane Potential Assay: Two days prior to the experiment, frozen HEK293 cells stably expressing recombinant human Nav1.7 were quickly thawed and plated at 25,000 cells/well in growth medium [DMEM (Invitrogen #11965) with 10% HI FBS (Invitrogen #10082), 2 mM glutamine, 100 units/mL penicillin, 0.1 mg/mL streptomycin (PSG, Sigma #G1146), and 500 ug/mL Geneticin (Invitrogen #10131)] in black-walled, clear-bottom 384-well poly-D-lysine-coated assay plates (Greiner Bio-One, Frickenhausen, Germany) and incubated in a humidified 5% CO2 incubator at 37° C. On the day of the assay, medium was removed by aspiration, and cells were washed with assay buffer [HBSS (Invitrogen, Carlsbad, Calif.) containing 20 mM HEPES (Invitrogen, Carlsbad, Calif.)]. After washing, 30 uL assay buffer containing the fluorescent voltage-sensor probe CC2-DMPE (Invitrogen, Carlsbad, Calif.) at 20 uM and 0.01% pluronic F-127 (Invitrogen, Carlsbad, Calif.) was added to the cells. Cells were incubated for 40 minutes at room temperature in the dark. Following the incubation, the cells were washed and 30 uL assay buffer containing 2.5 uM DiSBAC2(3) substrate (Invitrogen, Carlsbad, Calif.) and 0.5 mM VABSC-1 (Invitrogen, Carlsbad, Calif.) was added to the cells. The cells were incubated for 90 minutes at room temperature in the dark. Fluorescence readings were made using a FLIPR TETRA (Molecular Devices, Sunnyvale Calif.) equipped with voltage-sensor probe optics. At the start of each experiment the optimal (EC80) concentration of depolarizing agent (veratridine) was determined by testing a dilution curve of assay buffer containing veratridine (Sigma-Aldrich, St. Louis, Mo.) and 1 mg/mL scorpion venom (SVqq, from Leiurus quinquestriatus; Sigma-Aldrich, St. Louis, Mo.). Compounds were dissolved in dimethyl sulfoxide, and 8-point, 1:3 dilution concentration-response curves were prepared in duplicate in dimethyl sulfoxide, followed by preparation of 0.8 uL/well daughter plates of the dilutions. Test compounds in the daughter plate were diluted to (Ë¿3x) solutions in assay buffer immediately before assaying. Using the FLIPR TETRA, 20 uL of the (3x) compound solutions were first added to the cells, then 20 uL of depolarizing solution (3xEC80 veratridine+SVqq) were added 3 minutes later to activate the channel. Changes in fluorescence were measured at wavelengths of 440-480 nm and 565-625 nm over the course of the experimental run. Membrane depolarization was expressed as a ratio of the maximum F440-480 nm/F565-625 nm reading above average baseline F440-480 nm/F565-625 nm reading. IC50 values were calculated from curve fits of the ratio data using a four-parameter logistic Hill equation (Accelrys Assay Explorer 3.3 Client, Accelrys, San Diego, Calif.) with percent inhibition plotted against compound concentration.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3889155.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3889155/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.70\n",
      "      max pchembl_value: 6.78\n",
      "      median pchembl_value: 5.60\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3707623:\n",
      "    description: \"Assay CHEMBL3707623: HTRF Assay: The ability of compounds to inhibit the activity of JAK1, JAK2, JAK3, and Tyk2 was measured using a recombinant purified GST-tagged catalytic domain for each enzyme (Invitrogen JAK1 #M4290, JAK2 #M4290, JAK3 #M4290, Tyk2 #M4290) in an HTRF format biochemical assay. The reactions employed a common peptide substrate, LCB-EQEDEPEGDYFEWLW-NH2 (in-house). The basic assay protocol is as follows: First, 250 mL of diluted compounds in DMSO were dispensed into the wells of a dry 384-well Black plate (Greiner #781076) using a Labcyte Echo 555 acoustic dispenser. Subsequent reagent additions employed an Agilent Bravo. Next, 18 uL of 1.11x enzyme and 1.11x substrate in 1x assay buffer (Invitrogen kinase buffer #PV3189, 2 mM DTT, 0.05% BSA) were added to the wells and shaken and then preincubated for 30 minutes at ambient temperature to allow compound binding to equilibrate. After equilibration, 2 uL of 10xATP in 1x assay buffer was added to initiate the kinase reaction.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3707623.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3707623/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.82\n",
      "      max pchembl_value: 9.18\n",
      "      median pchembl_value: 7.58\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL1267245:\n",
      "    description: \"Assay CHEMBL1267245: Antiplasmodial activity against Plasmodium falciparum 3D7 after 72 hrs by SYBR green assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL1267245.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL1267245/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.70\n",
      "      max pchembl_value: 10.52\n",
      "      median pchembl_value: 5.00\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705647:\n",
      "    description: \"Assay CHEMBL3705647: Inhibition Assay: The stable cell line expressing human SGLT1 was seeded at 5x104 cells/well on BioCoat Poly-D-Lysine 96 well plate with Lid (Becton Dickinson and Company) and cultured at 37 C. under 5% CO2 overnight. The medium was replaced with 100 uL/well of a Na (+) buffer (140 mM choline chloride, 2 mM KCl, 1 mM MgCl2, 1 mM CaCl2, 10 mM HEPES, 5 mM Tris, pH7.4) followed by incubation for 20 minutes at 37 C. under 5% CO2. After removing the Na (+) buffer, a test compound solution which was prepared from Na (+) buffer (140 mM NaCl, 2 mM KCl, 1 mM MgCl2, 1 mM CaCl2, 10 mM HEPES, 5 mM Tris, pH7.4) containing BSA was added thereto at 40 uL/well. In addition, Na (+) buffer containing 8 kBq of 14C-AMG and 2 mM AMG was added thereto at 40 uL/well, and was mixed well. Na (+) buffer containing BSA was added to a blank well at 40 uL/well and additionally adding a Na (+) buffer containing 8 kBq of 14C-AMG and 2 mM AMG, and was mixed well.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705647.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705647/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.70\n",
      "      max pchembl_value: 9.15\n",
      "      median pchembl_value: 7.89\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3887033:\n",
      "    description: \"Assay CHEMBL3887033: Enzyme Assay: Isolated TRK Enzyme assays use the HTRF KinEASE-TK kit (Cisbio Cat# 62TK0PEJ) with recombinant His-tagged cytoplasmic domains of each TRK receptor sourced from Invitrogen (see table below). This activity-assay measures the phosphorylation of tyrosine residues within a substrate from the HTRF kit which has been validated by Cisbio for a variety of tyrosine kinases including the TRK receptors. Assay details: Target:TRKA; Invitrogen Cat# PV3144; Amino acids:aa 441-796; FAC enzyme:4nM; FAC ATP:40uM; Assay Reaction Time:35min.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3887033.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3887033/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.01\n",
      "      max pchembl_value: 9.17\n",
      "      median pchembl_value: 6.68\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3888966:\n",
      "    description: \"Assay CHEMBL3888966: In Vitro Assay: CHO-K1 CXCR7 b-arrestin cells are detached from culture dishes with a cell dissociation buffer (Invitrogen, #13151-014) and collected in growing medium (F12 HAMS 90% (v/v)/FCS 10% (v/v), Penicilin/streptomycin 1% (v/v)). 5000 cells per well (in 20 μl) are seeded in a 384 well plate (white-walled, clear bottom; BD Falcon #353274). The plate is incubated at 37° C./5% CO2 for 24 hours. Medium is then replaced by 20 μl OPTIMEM (Invitrogen #31985) for 3 to 4 hours. Test compounds are dissolved at 10 mM in DMSO and serially diluted in DMSO to 200× of the final concentration for dose response curves. Compounds are then diluted 1:33.3 in HBSS1×. 5 μl/well of HBSS1×/20 mM HEPES/0.2% BSA are added to the assay plate followed by addition of 5 μl/well of diluted compounds. CXCL12 (Peprotech #300-28A) may be used as a reference agonist. The plate is incubated for 90 minutes at 37° C. 12 μl of detection reagent (Path Hunter Detection Kit, DiscoveRx, #93-0001) is transferred to the assay plate and to the plate is incubated for 1 hour at room temperature. Luminescent signal is read in a microplate reader (FLUOstar Optima, bmg).\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3888966.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3888966/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: EC50\n",
      "      min pchembl_value: 5.56\n",
      "      max pchembl_value: 9.40\n",
      "      median pchembl_value: 7.92\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL1267250:\n",
      "    description: \"Assay CHEMBL1267250: Antiplasmodial activity against Plasmodium falciparum HB3 after 72 hrs by SYBR green assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL1267250.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL1267250/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.80\n",
      "      max pchembl_value: 10.22\n",
      "      median pchembl_value: 5.00\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705858:\n",
      "    description: \"Assay CHEMBL3705858: Spectrophotometric 384 Well Assay: Malonyl CoA formation by acetyl CoA carboxylases is stoichometrically linked to the consumption of ATP. ACC2 activity is measured in a NADH-linked kinetic method measuring ADP generated during the ACC reaction using a coupled lactate dehydrogenase/pyruvate kinase reaction.For biological testing, a human ACC2 construct which lacks the 128 amino acids at the N-terminus for increased solubility (nt 385-6966 in Genbank entry AJ575592) is cloned. The protein is then expressed in insect cells using a baculoviral expression system. Protein purification is performed by anion exchange.All compounds are dissolved in dimethyl sulfoxide (DMSO) to a concentration of 10 mM.Assay reactions are then carried out in 384-well plates, with hACC2 in an appropriate dilution and at final assay concentrations (f.c.) of 100 mM Tris (pH 7.5), 10 mM trisodium citrate, 25 mM KHCO3, 10 mM MgCl2, 0.5 mg/ml BSA, 3.75 mM reduced L-glutathione, 15 U/ml lactate dehydrogenase, 0.5 mM phosphoenolpyruvate.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705858.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705858/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.54\n",
      "      max pchembl_value: 7.52\n",
      "      median pchembl_value: 6.66\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705960:\n",
      "    description: \"Assay CHEMBL3705960: In Vitro Assay: The Cdk4 and Cdk6 inhibitory activity of the compounds is measured with a kinase inhibition assay using recombinant Cdk4/CyclinD1 or Cdk6/CyclinD3 protein complexes. The protein substrate used in the assay is the retinoblastoma protein (Rb). The kinase reactions are carried out in a 96-well filter plate (MSDV N6B50, Millipore). Compounds are serially diluted in kinase buffer (20 mM Tris-HCl, pH 7.4, 50 mM NaCl, 1 mM DTT, 1 mg/ml BSA) and added to the reaction mixture containing 2.5 ng/ml Cdk4/CyclinD1 or Cdk6/CyclinD3, 25 μM ATP, 10 μCi/ml [33P]-ATP, 0.1 μg/ml Rb in the kinase buffer. The mixture is incubated at room temperature for 1 hour and the proteins precipitated with an equal volume of 20% TCA. The plates are washed with 10% TCA according to the manufacturer's instruction and dried at room temperature. The amount of the phosphorylated Rb is determined with a TopCount (PerkinElmer).\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705960.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705960/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.00\n",
      "      max pchembl_value: 9.89\n",
      "      median pchembl_value: 7.92\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705927:\n",
      "    description: \"Assay CHEMBL3705927: Spectrophotometric 384 Well Assay: Malonyl CoA formation by acetyl CoA carboxylases is stoichometrically linked to the consumption of ATP. ACC2 activity is measured in a NADH-linked kinetic method measuring ADP generated during the ACC reaction using a coupled lactate dehydrogenase/pyruvate kinase reaction.For biological testing, a human ACC2 construct which lacks the 128 amino acids at the N-terminus for increased solubility (nt 385-6966 in Genbank entry AJ575592) is cloned. The protein is then expressed in insect cells using a baculoviral expression system. Protein purification is performed by anion exchange.All compounds are dissolved in dimethyl sulfoxide (DMSO) to a concentration of 10 mM.Assay reactions are then carried out in 384-well plates, with hACC2 in an appropriate dilution and at final assay concentrations (f.c.) of 100 mM Tris (pH 7.5), 10 mM trisodium citrate, 25 mM KHCO3, 10 mM MgCl2, 0.5 mg/ml BSA, 3.75 mM reduced L-glutathione, 15 U/ml lactate dehydrogenase, 0.5 mM phosphoenolpyruvate.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705927.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705927/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.53\n",
      "      max pchembl_value: 8.05\n",
      "      median pchembl_value: 6.78\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3887333:\n",
      "    description: \"Assay CHEMBL3887333: Enzyme Inhibition Assay: Measurements were performed using a modification of the scintillation proximity assay platform described previously by Georgopapadakou, N. H. et al. (22nd International Congress on Chemotherapy, 2001, Abstract P16.001). Compounds were solubilised in DMSO at a top concentration of 10 mM and serially diluted in half log steps to achieve a range of final assay concentrations of 100 uM to 1 nM. Compound at each concentration (100-fold final) was added to white 384 well plates in a volume of 0.5 ml. Human, A. fumigatus, T. brucei or L. major N-myristoyl transferase enzyme, dissolved to a working concentration of 10 nM in assay buffer (30 mM Tris/HCl pH 7.4, 0.5 mM EGTA, 0.5 mM EDTA, 1.25 mM DTT, 0.1% Triton X-100), was then added to columns 1 to 11 and 13 to 23 of the plates in a volume of 20 ml. To columns 12 and 24, 20 ml assay buffer was added to provide a no enzyme control. Following a 5 minute incubation at room temperature the substrates (GCGGSKVKPQPPQAK(Biotin)-Amide and myristoyl coenzyme A), dissolved in assay buffer, were added to all wells in a volume of 20 ml to start the reaction. The final concentrations of peptide and 3H-myristoyl coenzyme A were 0.5 mM and 125 nM respectively and the specific activity of the radiolabel was 8 Ci/mmol. Plates were then incubated at room temperature for up to 50 minutes (dependant upon the period of linearity for the different enzyme species) before SPA beads, suspended to 1 mg/ml in a stop solution (200 mM Phosphoric Acid/NaOH pH 4, 750 mM MgCl2), were added in a volume of 40 ml.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3887333.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3887333/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.02\n",
      "      max pchembl_value: 9.00\n",
      "      median pchembl_value: 6.17\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3888506:\n",
      "    description: \"Assay CHEMBL3888506: JAK Enzyme Assay: The activity of the isolated recombinant JAK1 and JAK2 kinase domain was measured by monitoring phosphorylation of a peptide derived from JAK3 (Val-Ala-Leu-Val-Asp-Gly-Tyr-Phe-Arg-Leu-Thr-Thr, fluorescently labeled on the N-terminus with 5-carboxyfluorescein) using the Caliper LabChip® technology (Caliper Life Sciences, Hopkinton, Mass.). To determine inhibition constants (Ki), compounds were diluted serially in DMSO and added to 50 μL kinase reactions containing purified enzyme (1.5 nM JAK1, or 0.2 nM JAK2), 100 mM HEPES buffer (pH 7.2), 0.015% Brij-35, 1.5 μM peptide substrate, ATP (25 μM), 10 mM MgCl2, 4 mM DTT at a final DMSO concentration of 2%. Reactions were incubated at 22° C. in 384-well polypropylene microtiter plates for 30 minutes and then stopped by addition of 25 μL of an EDTA containing solution (100 mM HEPES buffer (pH 7.2), 0.015% Brij-35, 150 mM EDTA), resulting in a final EDTA concentration of 50 mM. After termination of the kinase reaction, the proportion of phosphorylated product was determined as a fraction of total peptide substrate using the Caliper LabChip® 3000 according to the manufacturer's specifications.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3888506.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3888506/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: Ki\n",
      "      min pchembl_value: 6.92\n",
      "      max pchembl_value: 10.22\n",
      "      median pchembl_value: 9.08\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3888505:\n",
      "    description: \"Assay CHEMBL3888505: JAK Enzyme Assay: The activity of the isolated recombinant JAK1 and JAK2 kinase domain was measured by monitoring phosphorylation of a peptide derived from JAK3 (Val-Ala-Leu-Val-Asp-Gly-Tyr-Phe-Arg-Leu-Thr-Thr, fluorescently labeled on the N-terminus with 5-carboxyfluorescein) using the Caliper LabChip® technology (Caliper Life Sciences, Hopkinton, Mass.). To determine inhibition constants (Ki), compounds were diluted serially in DMSO and added to 50 μL kinase reactions containing purified enzyme (1.5 nM JAK1, or 0.2 nM JAK2), 100 mM HEPES buffer (pH 7.2), 0.015% Brij-35, 1.5 μM peptide substrate, ATP (25 μM), 10 mM MgCl2, 4 mM DTT at a final DMSO concentration of 2%. Reactions were incubated at 22° C. in 384-well polypropylene microtiter plates for 30 minutes and then stopped by addition of 25 μL of an EDTA containing solution (100 mM HEPES buffer (pH 7.2), 0.015% Brij-35, 150 mM EDTA), resulting in a final EDTA concentration of 50 mM. After termination of the kinase reaction, the proportion of phosphorylated product was determined as a fraction of total peptide substrate using the Caliper LabChip® 3000 according to the manufacturer's specifications.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3888505.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3888505/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: Ki\n",
      "      min pchembl_value: 7.02\n",
      "      max pchembl_value: 9.82\n",
      "      median pchembl_value: 9.17\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3887887:\n",
      "    description: \"Assay CHEMBL3887887: In Vitro Assay: Intracellular Calcium Measurements : In Vitro Assay: Intracellular Calcium Measurements:Chinese hamster ovary (CHO) cells expressing the human orexin-1 receptor and the human orexin-2 receptor, respectively, are grown in culture medium (Ham F-12 with L-Glutamine) containing 300 g/ml G418, 100 U/ml penicillin, 100 g/ml streptomycin and 10% heat inactivated fetal calf serum (FCS). The cells are seeded at 20,000 cells/well into 384-well black clear bottom sterile plates (Greiner). The seeded plates are incubated overnight at 37° C. in 5% CO2. Human orexin-A as an agonist is prepared as 1 mM stock solution in MeOH:water (1:1), diluted in HBSS containing 0.1% bovine serum albumin (BSA), NaHCO3: 0.375 g/l and 20 mM HEPES for use in the assay at a final concentration of 3 nM.Antagonists are prepared as 10 mM stock solution in DMSO, then diluted in 384-well plates using DMSO followed by a transfer of the dilutions into in HBSS containing 0.1% bovine serum albumin (BSA), NaHCO3: 0.375 g/l and 20 mM HEPES. On the day of the assay, 50 μl of staining buffer (HBSS containing 1% FCS, 20 mM HEPES, NaHCO3: 0.375 g/l, 5 mM probenecid (Sigma) and 3 M of the fluorescent calcium indicator fluo-4 AM (1 mM stock solution in DMSO, containing 10% pluronic) is added to each well. The 384-well cell-plates are incubated for 50 min at 37° C. in 5% CO2 followed by equilibration at RT for 30 min before measurement.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3887887.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3887887/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.16\n",
      "      max pchembl_value: 9.00\n",
      "      median pchembl_value: 6.96\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3887886:\n",
      "    description: \"Assay CHEMBL3887886: In Vitro Assay: Intracellular Calcium Measurements : In Vitro Assay: Intracellular Calcium Measurements:Chinese hamster ovary (CHO) cells expressing the human orexin-1 receptor and the human orexin-2 receptor, respectively, are grown in culture medium (Ham F-12 with L-Glutamine) containing 300 g/ml G418, 100 U/ml penicillin, 100 g/ml streptomycin and 10% heat inactivated fetal calf serum (FCS). The cells are seeded at 20,000 cells/well into 384-well black clear bottom sterile plates (Greiner). The seeded plates are incubated overnight at 37° C. in 5% CO2. Human orexin-A as an agonist is prepared as 1 mM stock solution in MeOH:water (1:1), diluted in HBSS containing 0.1% bovine serum albumin (BSA), NaHCO3: 0.375 g/l and 20 mM HEPES for use in the assay at a final concentration of 3 nM.Antagonists are prepared as 10 mM stock solution in DMSO, then diluted in 384-well plates using DMSO followed by a transfer of the dilutions into in HBSS containing 0.1% bovine serum albumin (BSA), NaHCO3: 0.375 g/l and 20 mM HEPES. On the day of the assay, 50 μl of staining buffer (HBSS containing 1% FCS, 20 mM HEPES, NaHCO3: 0.375 g/l, 5 mM probenecid (Sigma) and 3 M of the fluorescent calcium indicator fluo-4 AM (1 mM stock solution in DMSO, containing 10% pluronic) is added to each well. The 384-well cell-plates are incubated for 50 min at 37° C. in 5% CO2 followed by equilibration at RT for 30 min before measurement.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3887886.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3887886/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.75\n",
      "      max pchembl_value: 9.00\n",
      "      median pchembl_value: 6.60\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3887796:\n",
      "    description: \"Assay CHEMBL3887796: IMAP-FP Assay: Activated ERK2 activity was determined in an IMAP-FP assay (Molecular Devices). Using this assay format, the potency (IC50) of each compound was determined from a 10 point (1:3 serial dilution, 3 uM starting compound concentration) titration curve using the following outlined procedure. To each well of a black Corning 384-well plate (Corning Catalog #3575), 7.5 nL of compound (3333 fold dilution in final assay volume of 25 uL) was dispensed, followed by the addition of 15 uL of kinase buffer (tween containing kinase buffer, Molecular Devices) containing 0.0364 ng/mL (0.833 nM) of phosphorylated active hERK2 enzyme. Following a 15 minute compound & enzyme incubation, each reaction was initiated by the addition of 10 uL kinase buffer containing 2.45 uM ERK2 IMAP substrate peptides (2.25 uM-unlabeled IPTTPITTTYFFFK-COOH and 200 nM-labeled IPTTPITTTYFFFK-SFAM (5-carboxyfluorescein)-COOH), and 75 uM ATP.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3887796.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3887796/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.52\n",
      "      max pchembl_value: 9.52\n",
      "      median pchembl_value: 8.10\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3887757:\n",
      "    description: \"Assay CHEMBL3887757: Enzyme Assay: JAK1, JAK2, JAK3 and Tyk2 were purchased from Carna Biosciences, Inc. As the substrate, LANCE Ultra ULight-JAK1 Peptide (manufactured by PerkinElmer Co., Ltd. (PE)) was used. Dilute solutions of compounds and enzymes in assay buffer (50 mM HEPES pH7.5, 1 mM EGTA, 1 mM MgCl2, 2 mM DTT, 0.01% Tween20) were dispensed into wells of a 384-well black plate. After 5 minutes of preincubation, dilute solutions of the substrate and ATP (adenosine triphosphate) were added at a final concentration of 100 μM, and the plate was incubated at room temperature for 2 hours. After addition of a termination reagent containing EDTA (ethylenediamine tetraacetic acid) at a final concentration of 10 mM, LANCE Eu-W1024 Anti-phosphotyrosine (PT66) (manufactured by PE) was added, and after 1 hour of incubation, the fluorescences were measured with ARVO-HTS.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3887757.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3887757/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 6.11\n",
      "      max pchembl_value: 9.96\n",
      "      median pchembl_value: 8.03\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL1826790:\n",
      "    description: \"Assay CHEMBL1826790: Inhibition of human c-MET\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL1826790.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL1826790/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.00\n",
      "      max pchembl_value: 9.30\n",
      "      median pchembl_value: 7.17\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3887849:\n",
      "    description: \"Assay CHEMBL3887849: IMAP-FP Assay: Activated ERK2 activity was determined in an IMAP-FP assay (Molecular Devices). Using this assay format, the potency (IC50) of each compound was determined from a 10 point (1:3 serial dilution, 3 uM starting compound concentration) titration curve using the following outlined procedure. To each well of a black Corning 384-well plate (Corning Catalog #3575), 7.5 nL of compound (3333 fold dilution in final assay volume of 25 uL) was dispensed, followed by the addition of 15 uL of kinase buffer (tween containing kinase buffer, Molecular Devices) containing 0.0364 ng/mL (0.833 nM) of phosphorylated active hERK2 enzyme. Following a 15 minute compound & enzyme incubation, each reaction was initiated by the addition of 10 uL kinase buffer containing 2.45 uM ERK2 IMAP substrate peptides (2.25 uM-unlabeled IPTTPITTTYFFFK-COOH and 200 nM-labeled IPTTPITTTYFFFK-SFAM (5-carboxyfluorescein)-COOH), and 75 uM ATP. The final reaction in each well of 25 uL consists of 0.5 nM hERK2, 900 nM unlabeled peptide, 80 nM labeled-peptide, and 30 uM ATP.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3887849.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3887849/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.61\n",
      "      max pchembl_value: 9.70\n",
      "      median pchembl_value: 8.38\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3880346:\n",
      "    description: \"Assay CHEMBL3880346: Inhibition of VEGFR2 (unknown origin) transfected in mouse BA/F3 cells assessed as decrease in cell proliferation in absence of mouse IL3 after 24 hrs by Alamar blue assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3880346.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3880346/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.99\n",
      "      max pchembl_value: 7.61\n",
      "      median pchembl_value: 5.54\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705868:\n",
      "    description: \"Assay CHEMBL3705868: Biochemical Assay: The FASN enzyme was isolated from SKBr3 cells. SKBr3 is a human breast cancer cell-line with high levels of FASN expression. It is estimated that FASN comprises about 25% of the cytosolic proteins in this cell line. SKBr3 cells were homogenized in a dounce homogenizer then centrifuged for 15 minutes at 4C. to remove particulate matter. The supernatant was then analyzed for protein content, diluted to the appropriate concentration, and used to measure FASN activity. The presence of FASN was confirmed by western blot analysis. A similar method for isolation of FASN from SKBr3 cells is described in Teresa, P. et al. (Clin. Cancer Res. 2009; 15(24), 7608-7615).FASN activity of the SKBr3 cell extract was determined by measuring either NADPH oxidation or the amount of thiol-containing coenzyme A (CoA) released during the fatty acid synthase reaction. The dye CPM (7-diethylamino-3-(4Ã¢-maleimidyl-phenyl)-4-methylcoumarin) contains a thiol reactive group that increases its fluorescence emission.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705868.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705868/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.37\n",
      "      max pchembl_value: 8.00\n",
      "      median pchembl_value: 6.76\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3887758:\n",
      "    description: \"Assay CHEMBL3887758: Enzyme Assay: JAK1, JAK2, JAK3 and Tyk2 were purchased from Carna Biosciences, Inc. As the substrate, LANCE Ultra ULight-JAK1 Peptide (manufactured by PerkinElmer Co., Ltd. (PE)) was used. Dilute solutions of compounds and enzymes in assay buffer (50 mM HEPES pH7.5, 1 mM EGTA, 1 mM MgCl2, 2 mM DTT, 0.01% Tween20) were dispensed into wells of a 384-well black plate. After 5 minutes of preincubation, dilute solutions of the substrate and ATP (adenosine triphosphate) were added at a final concentration of 100 μM, and the plate was incubated at room temperature for 2 hours. After addition of a termination reagent containing EDTA (ethylenediamine tetraacetic acid) at a final concentration of 10 mM, LANCE Eu-W1024 Anti-phosphotyrosine (PT66) (manufactured by PE) was added, and after 1 hour of incubation, the fluorescences were measured with ARVO-HTS.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3887758.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3887758/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.60\n",
      "      max pchembl_value: 9.00\n",
      "      median pchembl_value: 7.28\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL4649964:\n",
      "    description: \"Assay CHEMBL4649964: Calibr Malaria Delayed Death Assay SOP\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL4649964.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL4649964/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.55\n",
      "      max pchembl_value: 8.34\n",
      "      median pchembl_value: 6.13\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3887945:\n",
      "    description: \"Assay CHEMBL3887945: Radioligand Displacement Assay: Binding assay was performed in a final assay volume of 250 ul. First, 25 ul of test compound, previously diluted in Binding-Buffer (Binding-Buffer: 50 mM Tris-Base, 100 mM NaCl, 1 mM EDTA, 0.1% BSA (protease free), 0.01% NaN3, 10 mM MnCl2, pH 7.0) was placed into each well. After addition of 75 ul Binding-Buffer, 50 ul of the radioligand 3H-PGD2 (at 2.5 nM (220.000 dpm/well) from ANAWA ART0662) was added to each well. Binding assay was started by addition of 100 ul CRTH2 membrane fragments, reaching a final concentration of 20 ug/well. For non-specific binding, PGD2 was added to the reaction mixture to 10 mM final concentration. This assay mix was incubated for 90 minutes at room temperature and then filtered through a GF/C filter 96-well plate which was pre-soaked for 3 hours in 0.5% polyethyleneimine (PEI). The filter-wells were washed three times with ice cold Binding-Buffer. Then, 40 ul of Microscint-40 (Packard) was added to each well and the retained radioactivity quantified in a Topcount (Packard).\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3887945.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3887945/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 6.01\n",
      "      max pchembl_value: 9.70\n",
      "      median pchembl_value: 8.04\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705655:\n",
      "    description: \"Assay CHEMBL3705655: Alternative Enzyme Assay: The assay used AlphaScreen technology (Gray et al., Analytical Biochemistry, 2003, 313: 234-245) to determine the ability of test compounds to inhibit phosphorylation by recombinant mTOR.Test compounds were prepared as 10 mM stock solutions in DMSO and diluted into water as required to give a range of final assay concentrations. Aliquots (2 ul) of each compound dilution were placed into a well of a Greiner 384-well low volume (LV) white polystyrene plate (Greiner Bio-one). A 10 ul mixture of recombinant purified mTOR enzyme, 1 uM biotinylated peptide substrate (Biotin-Ahx-Lys-Lys-Ala-Asn-Gln-Val-Phe-Leu-Gly-Phe-Thr-Tyr-Val-Ala-Pro-Ser-Val-Leu-Glu-Ser-Val-Lys-Glu-NH2; Bachem UK Ltd), ATP (20 uM) in a buffer solution [comprising Tris-HCl pH7.4 buffer (50 mM), EGTA (0.1 mM), bovine serum albumin (0.5 mg/ml), DTT (1.25 mM) and manganese chloride (10 mM)] were added to the assay plates and incubated with compound for 2 hours at room temperature.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705655.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705655/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.12\n",
      "      max pchembl_value: 9.82\n",
      "      median pchembl_value: 7.07\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3706369:\n",
      "    description: \"Assay CHEMBL3706369: Binding Assay: The efficacy of compounds of the invention in inhibiting the PI3K induced-lipid phosphorylation may be tested in the following binding assay. The assay combines the scintillation proximity assay technology (SPA, Amersham) with the capacity of neomycin (a polycationic antibiotic) to bind phospholipids with high affinity and specificity. The Scintillation Proximity Assay is based on the properties of weakly emitting isotopes (such as 3H, 125I, 33P). Coating SPA beads with neomycin allows the detection of phosphorylated lipid substrates after incubation with recombinant PI3K and radioactive ATP in the same well, by capturing the radioactive phospholipids to the SPA beads through their specific binding to neomycin. To a 384 wells MTP containing 5 ul of the test compound of Formula (I) (solubilized in 2% DMSO; to yield a final concentration of 20, 5, 1.25, 0.3125, 0.0781, 0.0195, 0.0049, 0.0012, 0.0003 and 0.00075 uM of the test compound).\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3706369.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3706369/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.75\n",
      "      max pchembl_value: 8.22\n",
      "      median pchembl_value: 6.30\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705971:\n",
      "    description: \"Assay CHEMBL3705971: In Vitro Enzyme Assay: Using the following procedure, varying concentration of compounds of the invention were assessed for their ability to inhibit c-Abl enzyme's phosphorylation of 5-FAM-EAIYAHPFAKKK-CONH2 (Caliper LifeSciences, cat #760346) peptide substrate in the presence of ATP at Km app. The phosphorylated product is detected using the Caliper mobility shift detection method where product and substrate are electrophoretically separated.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705971.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705971/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: Ki\n",
      "      min pchembl_value: 6.36\n",
      "      max pchembl_value: 10.73\n",
      "      median pchembl_value: 9.00\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3734233:\n",
      "    description: \"Assay CHEMBL3734233: Inhibition of N-terminally GST-tagged human DLK catalytic domain (1 to 520 amino acids) using N- terminal His-tagged MKK4 K131M substrate incubated for 60 mins by TR-FRET assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3734233.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3734233/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: Ki\n",
      "      min pchembl_value: 5.19\n",
      "      max pchembl_value: 9.51\n",
      "      median pchembl_value: 7.77\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL1267247:\n",
      "    description: \"Assay CHEMBL1267247: Antiplasmodial activity against Plasmodium falciparum D10 after 72 hrs by SYBR green assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL1267247.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL1267247/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.70\n",
      "      max pchembl_value: 10.70\n",
      "      median pchembl_value: 5.00\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3882755:\n",
      "    description: \"Assay CHEMBL3882755: Inhibition of human IGF1R kinase domain (M954 to C1367 residues) un-phosphorylated apo-enzyme form expressed in baculovirus based expression system using poly-EY peptide incubated for 2 hrs by luciferase-luciferin coupled chemiluminescent assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3882755.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3882755/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 6.30\n",
      "      max pchembl_value: 7.30\n",
      "      median pchembl_value: 6.30\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705323:\n",
      "    description: \"Assay CHEMBL3705323: DELFIA assay: The kinase activity is measured by DELFIA assay (dissociation-enhanced lanthanide fluorescence immunoassay, Perkin Elmer). The cytoplasmic kinase domain of human IGF-1R (amino acids 964-1370) is expressed as a fusion protein with a glutathione-S-transferase tag (IGF-1R-GST) in High Five Cells (Invitrogen). Enzyme activity is measured in the presence of substances and a control substance. Poly-glutamate-tyrosine peptide (pEY, Sigma Aldrich) and biotinylated pEY (bio-pEY) are used as reaction substrates. 10 uL of substance in 25% DMSO are mixed with 30 uL of IGF-1R-GST solution (67 mM HEPES pH 7.4, 15 ug/mL pEY, 1.7 ug/mL bio-pEY, 13.3 mM MgCl2, 3.3 mM dithiothreitol, 0.0033% Brij 35, 2 ng IGF-1R-GST) in 96-well plates. The reactions are started with 10 uL of a 750 uM ATP solution. After 40 min at RT the reactions are stopped with 50 uL of stop solution (250 mM EDTA, 20 mM HEPES pH 7.4).\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705323.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705323/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.53\n",
      "      max pchembl_value: 9.74\n",
      "      median pchembl_value: 8.10\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3889082:\n",
      "    description: \"Assay CHEMBL3889082: Fluorescence Biochemical Assay: The IDH1 (R132H) mutant catalyzes the reduced form of NADP+ (NADPH) and α-ketoglutarate (α-KG) to form nicotinamide adenine dinucleotide phosphate (NADP+) and R (−)-2-hydroxyglutarate (2HG). The reaction can be monitored kinetically by following the oxidation of NADPH to NADP+ which is measured using fluorescence, excitation at 355 nm and emission at 530 nm. Reactions were monitored using the Perkin-Elmer Envision, Model 2101. More specifically, the biochemical reactions were performed at room temperature in 384-well Greiner flat-bottom plates (Cat. No. 781076) using a final reaction volume of 20 μL and the following assay buffer conditions: 50 mM HEPES pH 7.5, 10 mM MgCl2, 1 mM DTT, 0.02% BSA, 0.02% Tween-20, 10 μM NADPH and 100 μM α-KG. The final reaction mixture contained 2.5% DMSO and test compounds with concentrations ranging 0.0000008-25 μM. The IDH1 (R132H) enzyme was used at a final concentration of 10 nM.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3889082.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3889082/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.73\n",
      "      max pchembl_value: 9.00\n",
      "      median pchembl_value: 6.97\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL1119333:\n",
      "    description: \"Assay CHEMBL1119333: Inhibition of AChE\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL1119333.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL1119333/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.00\n",
      "      max pchembl_value: 10.70\n",
      "      median pchembl_value: 7.00\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705983:\n",
      "    description: \"Assay CHEMBL3705983: Inhibition Assay: The activated cathepsin A was diluted in assay buffer (25 mM MES, pH 5.5, containing 5 mM DTT) and mixed with the test compound (dissolved in assay buffer containing (v/v) 3% DMSO) or, in the control experiments, with the vehicle in a multiple assay plate. After incubation for 15 min at room temperature, as substrate then bradykinin carrying an N-terminal ®Bodipy FL (4,4-difluoro-5,7-dimethyl-4-bora-3a,4a-diaza-s-indacene-3-propionyl) label (JPT Peptide Technologies GmbH; dissolved in assay buffer) was added to the mixture. The final concentration of cathepsin A was 833 ng/ml and the final concentration of labeled bradykinin 2 μM. After incubation for 15 min at room temperature the reaction was stopped by the addition of stop buffer (130 mM 2-(4-(2-hydroxy-ethyl)-piperazin-1-yl)-ethanesulfonic acid, pH 7.4, containing (v/v) 0.013% ®Triton X-100, 0.13% Coating Reagent 3 (Caliper Life Sciences), 6.5% DMSO and 20 μM ebelactone B (Sigma, #E0886)).\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705983.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705983/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.81\n",
      "      max pchembl_value: 7.53\n",
      "      median pchembl_value: 5.74\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3734174:\n",
      "    description: \"Assay CHEMBL3734174: Antagonist activity against human PAR4 expressed in HEK293 cells assessed as reduction in H-Ala-Phe(4-F)-Pro-Gly-Trp-Leu-Val-Lys-Asn-Gly-NH2-induced intracellular calcium mobilization incubated for 30 mins before agonist addition by FLIPR assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3734174.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3734174/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.33\n",
      "      max pchembl_value: 9.49\n",
      "      median pchembl_value: 7.38\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3887976:\n",
      "    description: \"Assay CHEMBL3887976: PIM Kinase Binding Activity Assay: PIM-1, PIM-2, and PIM-3 enzymes were generated as fusion proteins expressed in bacteria and purified by IMAC column chromatography (Sun, X., Chiu, J. F., and He, Q. Y. (2005) Expert Rev. Proteomics, 2:649-657). A fluorescent-labeled Pim-specific peptide substrate, was custom synthesized by American Peptide Company (Sunnyvale, Calif.). Reaction Buffer contained 10 mM HEPES, pH 7.2, 10 mM MgCl2, 0.01% Tween 20, 2 mM DTT. Termination Buffer contained 190 mM HEPES, pH 7.2, 0.015% Brij-35, 0.2% Coating Reagent 3 (Caliper Life Sciences, Hopkinton, Mass.), 20 mM EDTA. Separation Buffer contained 100 mM HEPES, pH 7.2, 0.015% Brij-35, 0.1% Coating Reagent 3, 1:200 Coating Reagent 8 (Caliper Life Sciences, Hopkinton, Mass.), 10 mM EDTA and 5% DMSO.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3887976.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3887976/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: Ki\n",
      "      min pchembl_value: 5.26\n",
      "      max pchembl_value: 11.00\n",
      "      median pchembl_value: 9.57\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3887296:\n",
      "    description: \"Assay CHEMBL3887296: P70S6K Enzyme Assay: P70S6K inhibitor compounds are diluted and plated in 96 well plates. A reaction mixture including the following components is then added to the compound plate to initiate the enzyme reaction; P70S6K (3 nM, T412E mutant, Millipore) is mixed with 24 uM ATP in an assay buffer containing 100 mM Hepes (pH 7.5), 5 mM MgCl2, 1 mM DTT, 0.015% Brij and 1 uM of the substrate peptide FITC-AHA-AKRRRLSSLRA-OH (derived from the S6 ribosomal protein sequence, FITC=fluorescein isothiocyanate, AHA=6-aminohexanoic acid). The reaction is incubated for 90 min at 25.degree. C., before the addition of 10 mM EDTA to stop the reaction. The proportion of substrate and product (phosphorylated) peptide is analysed on a Caliper Life Sciences Lab Chip 3000, using a pressure of -1.4 psi, and upstream and downstream voltages of -3000 and -700 respectively. Product peaks are resolved before substrate peaks on the resulting chromatograms.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3887296.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3887296/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.57\n",
      "      max pchembl_value: 10.22\n",
      "      median pchembl_value: 8.44\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705617:\n",
      "    description: \"Assay CHEMBL3705617: Radioligand Binding Assay: Radioligand Binding Studies: Saturation experiments. A representative compound of formula (I) was tritiated. Three tritiums were incorporated in place of methyl hydrogens to generate [3H]compound. Binding of this radioligand was preformed in 5 mL borosilicate glass test tubes at room temperature. Binding was initiated by adding membranes to increasing concentrations of [3H]compound in 100 mM NaCl, 20 mM Tris HCl, pH 7.4 buffer containing 0.01% w/v bovine serum albumin (BSA) for 18 h. Non-specific binding was determined in the presence of 1 uM unlabelled compound. After 18 h, the reactants were filtered through GF/C glass fiber filters presoaked in 0.5% w/v polyethylene imine. Filters were washed with 15 mL ice cold 100 mM NaCl, 20 mM Tris HCl, pH7.4 buffer containing 0.25% BSA to separate bound from free ligand. [3H]compound bound to filters was quantified by liquid scintillation counting.Competitive binding experiments.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705617.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705617/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.67\n",
      "      max pchembl_value: 9.00\n",
      "      median pchembl_value: 7.45\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3734552:\n",
      "    description: \"Assay CHEMBL3734552: Agonist activity at human recombinant ALX receptor expressed in HEK293 cells co-expressing G-protein Galpha16 assessed as effect on calcium response by Fluo-4-AM dye based FLIPR assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3734552.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3734552/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: EC50\n",
      "      min pchembl_value: 4.96\n",
      "      max pchembl_value: 10.15\n",
      "      median pchembl_value: 7.12\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3734553:\n",
      "    description: \"Assay CHEMBL3734553: Agonist activity at human recombinant FPRL2 receptor expressed in HEK293 cells co-expressing G-protein Galpha16 assessed as effect on calcium response by Fluo-4-AM dye based FLIPR assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3734553.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3734553/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: EC50\n",
      "      min pchembl_value: 4.64\n",
      "      max pchembl_value: 8.42\n",
      "      median pchembl_value: 6.13\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705790:\n",
      "    description: \"Assay CHEMBL3705790: LC-MS Biochemical Assay: Mutant IDH1 R132H catalytic activity was monitored using the quantitative liquid chromatography/mass spectrometry (LC-MS) detection of 2-HG, a product of the NADPH-dependent alpha-KG reduction reaction.More specifically, the biochemical reactions were performed at room temperature in 384-well Greiner flat-bottom plates (Costar, Cat. No. 781201) using a final reaction volume of 30 uL and the following assay buffer conditions: 50 mM HEPES pH 7.4, 10 mM MgCl2, 50 mM KCl, 1 mM DTT, 0.02% BSA, 5 uM NADPH and 100 uM alpha-KG.The final reaction mixture contained 3.3% DMSO and inhibitors with concentrations ranging 0.02-50 uM. The IDH1 enzyme was used at a final concentration of 0.25 nM. Following 45 minutes incubation, the reaction mixtures were quenched by the addition of 10 uL of 16% formic acid containing 800 nM of 5-carbon labeled 13C-2-HG). The protein was then precipitated by the addition of 2.5 volumes of acetonitrile followed by centrifugation (3000xg, 20 minutes).\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705790.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705790/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.40\n",
      "      max pchembl_value: 7.66\n",
      "      median pchembl_value: 6.35\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL4649962:\n",
      "    description: \"Assay CHEMBL4649962: Leishmania infantum Histidine tRNA synthetase (LdHisRS)\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL4649962.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL4649962/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.01\n",
      "      max pchembl_value: 6.64\n",
      "      median pchembl_value: 4.59\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3888087:\n",
      "    description: \"Assay CHEMBL3888087: cAMP Assay: CHO cells expressing human CB1 or CB2 receptors are seeded 17-24 hours prior to the experiment 50.000 cells per well in a black 96 well plate with flat clear bottom (Corning Costar #3904) in DMEM (Invitrogen No. 31331), 1.times.HT supplement, with 10% fetal calf serum and incubated at 5% CO2 and 37° C. in a humidified incubator. The growth medium was exchanged with Krebs Ringer Bicarbonate buffer with 1 mM IBMX and incubated at 30.degree. C. for 30 min Compounds were added to a final assay volume of 100 ul and incubated for 30 min at 30° C. Using the cAMP-Nano-TRF detection kit the assay (Roche Diagnostics) was stopped by the addition of 50 ul lysis reagent (Tris, NaCl, 1.5% Triton X100, 2.5% NP40, 10% NaN.sub.3) and 50 ul detection solutions (20 uM mAb Alexa700-cAMP 1:1, and 48 uM Ruthenium-2-AHA-cAMP) and shaken for 2 h at room temperature. The time-resolved energy transfer is measured by a TRF reader (Evotec Technologies GmbH), equipped with a ND:YAG laser as excitation source. The plate is measured twice with the excitation at 355 nm and at the emission with a delay of 100 ns and a gate of 100 ns, total exposure time 10 s at 730 (bandwidth 30 nm) or 645 nm (bandwidth 75 nm), respectively. The FRET signal is calculated.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3888087.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3888087/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: EC50\n",
      "      min pchembl_value: 5.69\n",
      "      max pchembl_value: 10.00\n",
      "      median pchembl_value: 7.32\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3880377:\n",
      "    description: \"Assay CHEMBL3880377: Inhibition of recombinant human fetal brain FLAG-tagged PRMT5/full length human testis 6His-tagged MEP50 expressed in baculovirus infected insect Sf9 cells using human histone H4 (1 to 15 residues) as substrate preincubated for 30 mins followed by [3H]-SAM addition\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3880377.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3880377/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.00\n",
      "      max pchembl_value: 7.00\n",
      "      median pchembl_value: 7.00\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705567:\n",
      "    description: \"Assay CHEMBL3705567: Biological Assay: The in vitro affinity of the compounds for the human orexin-1 and orexin-2 receptors was determined by competitive radioligand binding using [3H]SB SB674042 (1-(5-(2-fluoro-phenyl)-2-methyl-thiazol-4-yl)-1-((S)-2-(5-phenyl-(1,3,4)oxadiazol-2-ylmethyl)-pyrrolidin-1-yl)-methanone) (Langmead et al., British Journal of Pharmacology 2004, 141:340-346) and [3H]EMPA (N-ethyl-2[(6-methoxy-pyridin-3-yl)-(toluene-2-sulfonyl)-amino]-N-pyridin-3-ylmethyl acetamide) (Malherbe et al., British Journal of Pharmacology, 2009, 156(8), 1326-1341), respectively. The in vitro functional antagonism of the compounds on the human orexin-1 and orexin-2 receptors was determined using fluorometric imaging plate reader (FLIPR) based calcium assays.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705567.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705567/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: Ki\n",
      "      min pchembl_value: 5.00\n",
      "      max pchembl_value: 8.52\n",
      "      median pchembl_value: 6.76\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705699:\n",
      "    description: \"Assay CHEMBL3705699: In Vitro PX Assay: 293 cells stably transfected with human Nav1.7 were recorded in whole cell voltage clamp mode with the PatchXpress automated electrophysiology system (Molecular Devices, LLC, Sunnyvale, Calif.). Compound effects were measured on a partially inactivated state of the sodium channel. Cells were clamped to a holding potential yielding 20 to 50% inactivation. To elicit sodium current, channels were activated by pulsing to −10 mV for 20 msec. This voltage protocol was repeated at a rate of 0.1 Hz throughout the experiment. A single concentration of test compound was applied to cells for a duration of 3 minutes. Peak sodium current was measured at the end of the compound addition period to determine percent inhibition.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705699.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705699/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.43\n",
      "      max pchembl_value: 7.72\n",
      "      median pchembl_value: 6.16\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3887679:\n",
      "    description: \"Assay CHEMBL3887679: BACE Enzyme Assay (Ex.435): For each compound being tested, the BACE activity was monitored in a fluorescence quenching assay (FRET) using the ectodomain of BACE (aa 1-454) fused to a myc-his tag and secreted from HEK293/BACEect. cells into OptiMEM (Invitrogen) as enzyme source and a substrate peptide derived from the APP-Swedish mutation which possesses a Cy3-fluorophore at the N-terminus and a Cy5Q-quencher at the C-terminus (Cy3-SEVNLDAEFK-Cy5Q-NH2; Amersham). The substrate was dissolved at 1 mg/mL in DMSO.The assay was performed in the presence of 5 ul OptiMEM (supernatant collected over 24 hours and cleared from cellular debris by centrifugation) containing the ectodomain of BACE, 25 ul water containing the desired concentration of test compound and 1% DMSO, substrate peptide, 20 mM NaOAc, pH 4.4 and 0.04% Triton-X100 in a total assay volume of 50 ul in a 384 well plate. In general, 25 ul of compound dilution were given to the plate followed by the addition of 10 ul of BACE.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3887679.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3887679/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.52\n",
      "      max pchembl_value: 8.82\n",
      "      median pchembl_value: 7.38\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705480:\n",
      "    description: \"Assay CHEMBL3705480: Radioligand Binding Assay: Radioligand binding assays were carried out in a volume of 200 mL (96-well plates) which contained 100 mL of cell memebranes, [3H]flumazenil at a concentration of 1 nM for α1, α2, α3 subunits and 0.5 nM for α5 subunits and the test compound in the range of 10-10−3x10−6 M. Nonspecific binding was defined by 10−5 M diazepam and typically represented less than 5% of the total binding. Assays were incubated to equilibrium for 1 hour at 4 C. and harvested onto GF/C uni-filters (Packard) by filtration using a Packard harvester and washing with ice-cold wash buffer (50 mM Tris; pH 7.5). After drying, filter-retained radioactivity was detected by liquid scintillation counting.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705480.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705480/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: Ki\n",
      "      min pchembl_value: 7.05\n",
      "      max pchembl_value: 10.00\n",
      "      median pchembl_value: 8.08\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3887061:\n",
      "    description: \"Assay CHEMBL3887061: Time Resolved-Fluorescence Resonance Energy Transfer (TR-FRET) Assay: The inhibition constant (Ki) is the dissociation constant of an enzyme-inhibitor complex or a protein/small molecule complex, wherein the small molecule is inhibiting binding of one protein to another protein or peptide. Where the Ki for a compound is represented as > (greater than) a certain numerical value, it is intended to mean that the binding affinity value (e.g., for Bcl-XL) is greater than the limits of detection of the assay used. Where the binding selectivity ratio for a compound is represented as > (greater than) a certain numerical value, it is intended to mean that the selectivity of a particular compound for Bcl-2 over Bcl-XL is at least as great as the number indicated. Where the Ki for a compound is represented as < (less than) a certain numerical value, it is intended to mean that the binding affinity value (e.g., for Bcl-2) is lower than the limit of detection of the assay used. Inhibition constants were determined using Wang's equation (Wang Z-X).\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3887061.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3887061/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: Ki\n",
      "      min pchembl_value: 6.08\n",
      "      max pchembl_value: 11.00\n",
      "      median pchembl_value: 9.54\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3706310:\n",
      "    description: \"Assay CHEMBL3706310: LANCE ULight TR-FRET Kinase Assay: CDK9 enzyme activities were measured using LANCE ULight TR-FRET kinase assay reagents (PerkinElmer, Waltham, Mass.). Compounds were directly added in 100% DMSO to white low volume assay plates (Perkin Elmer Proxiplate 6008289) using a Labcyte Echo acoustic dispenser. Assay reagents in serine/threonine kinase assay buffer containing 20 mM HEPES, 10 mM MgCl2, 100 mM Na3VO4, and 0.0075% Triton X-100. were added for final reaction mixture concentrations of 1000 uM ATP, 100 nM U-light MBP peptide (Perkin Elmer TRF0109M) and reaction initiated with 4 nM CDK9/Cyclin T1 (Carna Biosciences 04-110). The kinase reaction was carried out for 30 minutes before addition of stopping buffer to a final of 20 mM EDTA and 0.5 nM of LANCE Ultra Europium anti-phospho-MBP antibody (PerkinElmer TRF0201M) in LANCE detection buffer (PerkinElmer CR97-100). The reaction was equilibrated for 1 hour and the signal read in the Perkin Elmer Envision in TR-FRET mode.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3706310.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3706310/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.72\n",
      "      max pchembl_value: 8.70\n",
      "      median pchembl_value: 7.35\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3887297:\n",
      "    description: \"Assay CHEMBL3887297: AKT Enzyme Assay: A TTP Mosquito liquid handling instrument is used to place 125 nl of the appropriate concentration of inhibitor in 100% DMSO (for a dose response curve calculation) into each well of a 384-well plate. To this reaction components are added to a final volume of 12.5 .mu.l: 0.1 ng/.mu.l His-AKT (Full Length), (Invitrogen, Part # P2999, Lot #641228C). 160 uM ATP (Fluka, 02055) 1 mM DTT (Sigma, D0632) 1 mM MgCl2 (Sigma, M1028) 1 .mu.M substrate peptide (sequence FITC-AHA-GRPRTSSFAEG-NH2), synthesized by Tufts Peptide Synthesis service. 100 mM HEPES pH 7.5 (Calbiochem, 391338)0.015% Brij-35 (Sigma, B4184)The reaction is incubated for 90 min at 25 C, and then stopped by the addition of 70 .mu.l of Stop buffer (100 mM HEPES pH 7.5, 0.015% Brij-35, 10 mM EDTA (Sigma, E7889)).The plate is read on a Caliper LC 3000 in an Off-Chip mobility shift assay format, using the following parameters for a 12-sipper chip: screening pressure -2.3 psi, upstream voltage -500, and downstream voltage -3000.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3887297.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3887297/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.82\n",
      "      max pchembl_value: 10.00\n",
      "      median pchembl_value: 7.70\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3888825:\n",
      "    description: \"Assay CHEMBL3888825: LanthaScreen™ Assay (5 mM ATP): a) 400 nl of a 1:2.15 serial dilution of test compound(98 μM top assay concentration) is spotted via Labcyte Echo to certain wells in a 384 well black, untreated plate. Control wells contain 400 nl of either DMSO or 400 nl of a known inhibitor in DMSO. b) 10 μl of a 2.5 nM LRRK2(G2019S mutation, GST-LRRK2(amino acids 970-2527)) enzyme solution in 1× assay buffer(50 mM Tris pH 8.5, 10 mM MgCl2, 0.01% Brij-35, 1.0 mM EGTA, 2 mM DTT, 0.05 mM NaVO4) is added to all wells. c) A 30 minute room temperature incubation is followed by addition of 10 μl of 800 nM fluorescein labeled LRRKtide peptide substrate and 10 mM ATP solution in 1× assay buffer to all wells. d) After a 35 minute room temperature incubation, 20 μl of TR-FRET Dilution Buffer(Invitrogen PV3756B) containing 4 nM Tb-labeled anti-phospho LRRKtide antibody and 20 mM EDTA is added to all wells. e) Plates are incubated at room temperature for 1 hour and read on an Envision™ multi-mode plate reader with LanthaScreen™ settings. Results are analysed using Assay Data Analyzer.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3888825.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3888825/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 6.04\n",
      "      max pchembl_value: 9.14\n",
      "      median pchembl_value: 7.98\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3706316:\n",
      "    description: \"Assay CHEMBL3706316: Fluorescence Resonance Energy Transfer Assay: The assay buffer used in this screen is 0.05 M acetate, pH 4.2, 10% DMSO final, 100 uM genapol (which is a nonionic detergent, below its Critical Micelle Concentration). The Beta Secretase enzyme (0.2 nM) is pre-incubated for one hour with inhibitors, typically in about 1 uL of DMSO according to a serial dilution, are added thereto. This assay is effectively started by the addition of FRET substrate (50 nM) and the combination is incubated for one hour. The FRET assay is terminated with by addition of Tris buffer, which raises the pH to neutrality, and the fluorescence is determined. The FRET substrate is a peptide with commercially available fluorophore and quencher, on opposite sides of the BACE cleavage site. Proteolytic cleavage of the FRET substrate releases quenching of fluorescence (excitation 488 nm and emission 425 nm).\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3706316.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3706316/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.39\n",
      "      max pchembl_value: 9.00\n",
      "      median pchembl_value: 6.98\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3888295:\n",
      "    description: \"Assay CHEMBL3888295: Time Resolved-Fluorescence Resonance Energy Transfer (TR-FRET) Assay: Representative compounds were serially diluted in dimethyl sulfoxide (DMSO) starting at 50 μM (2× starting concentration; 10% DMSO) and 10 μL were transferred into a 384-well plate. Then 10 μL of a protein/probe/antibody mix was added to each well at final concentrations listed in TABLE 1. The samples are then mixed on a shaker for 1 minute and incubated for an additional 3 hours at room temperature. For each assay, the probed antibody and protein/probe/antibody were included on each assay plate as negative and positive controls, respectively. Fluorescence was measured on the Envision (Perkin Elmer) using a 340/35 nm excitation filter and 520/525 (F-Bak peptide) and 495/510 nm (Tb-labeled anti-Histidine antibody) emission filters.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3888295.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3888295/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: Ki\n",
      "      min pchembl_value: 5.92\n",
      "      max pchembl_value: 11.00\n",
      "      median pchembl_value: 9.71\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3888209:\n",
      "    description: \"Assay CHEMBL3888209: TASK-1 Inhibiition Assay: Human TASK-1 channels were expressed in Xenopus oocytes. For this purpose, oocytes were isolated from Xenopus laevis and defoliculated. Subsequently, TASK-1-encoding RNA synthesized in vitro was injected into the oocytes. After two days of TASK-1 protein expression, TASK-1 currents were measured by two-microelectrode voltage clamp. Data were acquired and analyzed using a TEC-10cx amplifier (NPI Electronic, Tamm, Germany) connected to an ITC-16 interface (Instrutech Corp., Long Island, USA) and Pulse software (HEKA Elektronik, Lambrecht, Germany). Oocytes were clamped to -90 mV and TASK-1 mediated currents were measured during 500 ms voltage pulses to 40 mV. Oocytes were continuously superfused with ND96 buffer containing 96 mM sodium chloride, 2 mM potassium chloride, 1.8 mM calcium chloride, 1 mM magnesium chloride, 5 mM 4-(2-hydroxyethyl)-piperazine-1-ethanesulfonic acid (HEPES; pH adjusted to 7.4 with sodium hydroxide). All experiments were performed at room temperature.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3888209.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3888209/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.05\n",
      "      max pchembl_value: 7.89\n",
      "      median pchembl_value: 6.50\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705542:\n",
      "    description: \"Assay CHEMBL3705542: Time Resolved-Fluorescence Resonance Energy Transfer (TR-FRET) Assay: Representative compounds were serially diluted in dimethyl sulfoxide (DMSO) starting at 50 uM (2x starting concentration; 10% DMSO) and 10 uL were transferred into a 384-well plate. Then 10 uL of a protein/probe/antibody mix was added to each well at final concentrations listed in TABLE 1. The samples are then mixed on a shaker for 1 minute and incubated for an additional 3 hours at room temperature. For each assay, the probe/antibody and protein/probe/antibody were included on each assay plate as negative and positive controls, respectively. Fluorescence was measured on the Envision (Perkin Elmer) using a 340/35 nm excitation filter and 520/525 (F-Bak peptide) and 495/510 nm (Tb-labeled anti-Histidine antibody) emission filters. Inhibition constants (Ki) are shown in TABLE 2 below and were determined using Wang's equation (Wang Z.-X. An Exact Mathematical Expression For Describing Competitive Binding Of Two Different Ligands To A Protein Molecule.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705542.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705542/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: Ki\n",
      "      min pchembl_value: 5.92\n",
      "      max pchembl_value: 11.00\n",
      "      median pchembl_value: 9.71\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3887987:\n",
      "    description: \"Assay CHEMBL3887987: Biochemical Assay: PI3KA_Act: Genes encoding for full length p110alpha and p85alpha nSH-iSH2=niSH2 (p85a aminoacids 322-600) subunits of PI3Kalpha complex were subcloned from existing constructs into pFASTBAC Dual vector (Life Technologies, Carlsbad, Calif.) using standard cloning procedures. Gene encoding p110alpha subunit was subcloned into polyhedrine promoter while gene encoding p85alpha niSH2 domains was subcloned into p10 promoter. Additionally, Human Rhinovirus 3C Protease (HRV 3C) site was introduced between nSH2 and iSH2, replacing aminoacids 431-438 of p85alpha with LEVLFQGP HRV 3C recognition sequence, using standard QuickChange mutagenesis protocol (Agilent Technologies, CA). Recombinant baculovirus was generated using Bac-to-Bac protocol (Life Technologies, Carlsbad, Calif.). The biochemical assays of kinase activity of full-length PI3Kalpha (full-length p110alpha/p85a) or truncated PI3Kalpha (p110alpha/iSH2 p85a) were conducted using a fluorescence polarization format similar to the procedure of Yuan J., et al., (2011) PF-04691502, a Potent and Selective Oral Inhibitor of PI3K and mTOR Kinases with Antitumor Activity, Mol Cancer Ther. 10, 2189-2199.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3887987.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3887987/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: Ki\n",
      "      min pchembl_value: 6.93\n",
      "      max pchembl_value: 10.96\n",
      "      median pchembl_value: 9.90\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3707950:\n",
      "    description: \"Assay CHEMBL3707950: In Vitro Enzyme Assay: The ability of compounds to inhibit the kinase activity of baculovirus-expressed human FAK and JAK kinase using the time-resolved fluorescence (TRF) assay system.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3707950.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3707950/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.00\n",
      "      max pchembl_value: 9.57\n",
      "      median pchembl_value: 7.43\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705813:\n",
      "    description: \"Assay CHEMBL3705813: Inhibition Assay: The recombinant human DPPI was diluted in TAGZyme buffer to 1 U/ml (38.1 ug/ml, respectively), and then activated by mixing in a 1:2 ratio with a Cysteamine aqueous solution (2 mM) and incubating for 5 min at room temperature.Five uL test compound (final concentration 0.1 nM to 100 uM) in aqua bidest (containing 4% DMSO, final DMSO concentration 1%) were mixed with 10 uL of DPPI in MES buffer (final concentration 0.0125 ng/uL) and incubated for 10 min. Then, 5 uL of substrate in MES buffer (final concentration 50 uM) were added. The microtiter plates were then incubated at room temperature for 30 min. Then, the reaction was stopped by adding 10 uL of Gly-Phe-DMK in MES-buffer (final concentration 1 uM). The fluorescence in the wells was determined using a Molecular Devices SpectraMax M5 Fluorescence Reader (Ex 360 nm, Em 460 nm) or an Envision Fluorescence Reader (Ex 355 nm, Em 460 nm).\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705813.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705813/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.62\n",
      "      max pchembl_value: 9.22\n",
      "      median pchembl_value: 8.03\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3734252:\n",
      "    description: \"Assay CHEMBL3734252: Binding affinity to pyruvate dehydrogenase kinase 1 (unknown origin) incubated for 90 mins by VER-00160364 probe based fluorescence polarization assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3734252.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3734252/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.00\n",
      "      max pchembl_value: 8.00\n",
      "      median pchembl_value: 7.00\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705519:\n",
      "    description: \"Assay CHEMBL3705519: Reverse Mode Assay: The assay is based on the monitoring of intracellular Ca2+concentrations using the calcium-sensitive dye Fluo-4. CHO cells expressing NCX1 were loaded with the dye by means of the acetoxymethyl ester Fluo-4 AM (Invitrogen, F14202), which is cleaved intracellularly by esterase activity to yield the charged species of free Fluo-4. After an preincubation period with the test compound, Gramicidine (Sigma, G5002) was added. Gramicidine is an ionophor for Na+ions mediating an increase of intracellular Na+ions. Consequently, intracellular Na+ions are exchanged against extracellular Ca2+ions (Ca2+influx, reverse mode). The intracellular elevation of Ca2+ ions was detected by measuring the fluorescence of Fluo-4 at a wavelength of 520 nm by a FLIPR device.Briefly, for the reverse mode transport assay 18000 cells per well were seeded into a 96 well microplate (Corning COSTAR 3904) and incubated overnight in culture medium (1x Nut Mix F12 (Ham) (Gibco, 21765-029); 10% (v/v) fetal calf serum.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705519.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705519/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.60\n",
      "      max pchembl_value: 7.16\n",
      "      median pchembl_value: 6.40\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3707609:\n",
      "    description: \"Assay CHEMBL3707609: Biological Assay: The in vitro affinity of the compounds for the human orexin-1 and orexin-2 receptors was determined by competitive radioligand binding using [3H]SB SB674042 (1-(5-(2-fluoro-phenyl)-2-methyl-thiazol-4-yl)-1-((S)-2-(5-phenyl-(1,3,4)oxadiazol-2-ylmethyl)-pyrrolidin-1-yl)-methanone) (Langmead et al., British Journal of Pharmacology 2004, 141:340-346) and [3H]EMPA (N-ethyl-2[(6-methoxy-pyridin-3-yl)-(toluene-2-sulfonyl)-amino]-N-pyridin-3-ylmethyl acetamide) (Malherbe et al., British Journal of Pharmacology, 2009, 156(8), 1326-1341), respectively. The in vitro functional antagonism of the compounds on the human orexin-1 and orexin-2 receptors was determined using fluorometric imaging plate reader (FLIPR) based calcium assays.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3707609.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3707609/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: Ki\n",
      "      min pchembl_value: 5.00\n",
      "      max pchembl_value: 7.47\n",
      "      median pchembl_value: 5.54\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3706152:\n",
      "    description: \"Assay CHEMBL3706152: Enzyme Inhibition Assay: An IMAP TR-FRET assay was used to analyze the enzyme activity (Molecular Devices Corp., Sunnyvale Calif.). 5 μL of serial diluted PDE10A (BPS Bioscience, San Diego, Calif.) or tissue homogenate was incubated with equal volumes of diluted fluorescein labeled cAMP or cGMP for 60 min in 384-well polystyrene assay plates (Corning, Corning, N.Y.) at room temperature. After incubation, the reaction was stopped by adding 60 μL of diluted binding reagents and was incubated for 3 hours to overnight at room temperature. The plates were read on an Envision (Perkin Elmer, Waltham, Mass.) for time resolved fluorescence resonance energy transfer.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3706152.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3706152/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.00\n",
      "      max pchembl_value: 10.40\n",
      "      median pchembl_value: 7.10\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3887893:\n",
      "    description: \"Assay CHEMBL3887893: IMAP technology (Molecular Devices): A 20 ul reaction mixture contains 10 mM TriHCl, pH 7.2, 0.5 nM GST tagged IRAK4 (SignalChem), 100 nM fluorescent peptide substrate (RP7030, Molecular Devices), 100 uM ATP, 1 mM DTT, 1 mM MgCl2, and 0.01% Tween 20. The reaction is initiated by the addition of ATP. After incubation for 30 minutes at 25° C., 60 ul of Progressive IMAP Reagent (Molecular Devices) is added to stop the reaction. Change in RP7030's FP is determined by a FP reader (Analyst HT, LJL BioSystems).\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3887893.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3887893/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.05\n",
      "      max pchembl_value: 8.70\n",
      "      median pchembl_value: 6.96\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3888268:\n",
      "    description: \"Assay CHEMBL3888268: Bromodomain Domain Binding Assay: A time-resolved fluorescence resonance energy transfer (TR-FRET) assay was used to determine the affinities of compounds of the Examples listed in Table 1 for each bromodomain of human BRD4. His-tagged first (BD1: amino acids K57-E168) and second (BD2: amino acids E352-E168) bromodomains of human BRD4 were expressed and purified. An Alexa647-labeled BET-inhibitor was used as the fluorescent probe in the assay. Compound dilution series were prepared in DMSO via a 3-fold serial dilution from 2.5 mM to 42 nM. Compounds were then diluted 6:100 in assay buffer (20 mM Sodium Phosphate, pH 6.0, 50 mM NaCl, 1 mM Ethylenediaminetetraacetic acid disodium salt dihydrate, 0.01% Triton X-100, 1 mM DL-Dithiothreitol) to yield 3× working solutions. Six microliters (L) of the working solution was then transferred to white, low-volume assay plates (Costar #3673). A 1.5× assay mixture containing His-tagged bromodomain, Europium-conjugated anti-His antibody (Invitrogen PV5596) and the Alexa-647-conjugated probe molecule was also prepared. Twelve L of this solution were added to the assay plate to reach a final volume of 18 L. The final concentration of 1× assay buffer contains 2% DMSO, 50 M-0.85 nM compound, 8 nM His-tagged bromodomain, 1 nM Europium-conjugated anti-His-tag antibody and 100 nM or 30 nM probe (for BDI or BDII, respectively). After a one-hour incubation at room temperature, TR-FRET ratios were determined using an Envision multilabel plate reader (Ex 340, Em 495/520).\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3888268.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3888268/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: Ki\n",
      "      min pchembl_value: 5.36\n",
      "      max pchembl_value: 9.60\n",
      "      median pchembl_value: 7.98\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705306:\n",
      "    description: \"Assay CHEMBL3705306: In Vitro Enzyme Assay: The ability of compounds to inhibit the kinase activity of baculovirus-expressed human FAK and JAK kinase using the time-resolved fluorescence (TRF) assay system.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705306.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705306/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.00\n",
      "      max pchembl_value: 7.94\n",
      "      median pchembl_value: 5.70\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3888267:\n",
      "    description: \"Assay CHEMBL3888267: Bromodomain Domain Binding Assay: A time-resolved fluorescence resonance energy transfer (TR-FRET) assay was used to determine the affinities of compounds of the Examples listed in Table 1 for each bromodomain of human BRD4. His-tagged first (BD1: amino acids K57-E168) and second (BD2: amino acids E352-E168) bromodomains of human BRD4 were expressed and purified. An Alexa647-labeled BET-inhibitor was used as the fluorescent probe in the assay. Compound dilution series were prepared in DMSO via a 3-fold serial dilution from 2.5 mM to 42 nM. Compounds were then diluted 6:100 in assay buffer (20 mM Sodium Phosphate, pH 6.0, 50 mM NaCl, 1 mM Ethylenediaminetetraacetic acid disodium salt dihydrate, 0.01% Triton X-100, 1 mM DL-Dithiothreitol) to yield 3× working solutions. Six microliters (L) of the working solution was then transferred to white, low-volume assay plates (Costar #3673). A 1.5× assay mixture containing His-tagged bromodomain, Europium-conjugated anti-His antibody (Invitrogen PV5596) and the Alexa-647-conjugated probe molecule was also prepared. Twelve L of this solution were added to the assay plate to reach a final volume of 18 L. The final concentration of 1× assay buffer contains 2% DMSO, 50 M-0.85 nM compound, 8 nM His-tagged bromodomain, 1 nM Europium-conjugated anti-His-tag antibody and 100 nM or 30 nM probe (for BDI or BDII, respectively). After a one-hour incubation at room temperature, TR-FRET ratios were determined using an Envision multilabel plate reader (Ex 340, Em 495/520).\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3888267.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3888267/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: Ki\n",
      "      min pchembl_value: 5.02\n",
      "      max pchembl_value: 9.41\n",
      "      median pchembl_value: 7.85\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705852:\n",
      "    description: \"Assay CHEMBL3705852: IMAP TR-FRET assay: Enzyme Activity. An IMAP TR-FRET assay was used to analyze the enzyme activity (Molecular Devices Corp., Sunnyvale Calif.). 5 uL of serial diluted PDE10A (BPS Bioscience, San Diego, Calif.) or tissue homogenate was incubated with equal volumes of diluted fluorescein labeled cAMP or cGMP for 60 min in 384-well polystyrene assay plates (Corning, Corning, N.Y.) at room temperature. After incubation, the reaction was stopped by adding 60 uL of diluted binding reagents and was incubated for 3 hours to overnight at room temperature. The plates were read on an Envision (Perkin Elmer, Waltham, Mass.) for time resolved fluorescence resonance energy transfer. The data were analyzed with GraphPad Prism (La Jolla, Calif.).Enzyme Inhibition. To check the inhibition profile, 5 uL of serial diluted compounds were incubated with 5 uL of diluted PDE10 enzyme (BPS Bioscience, San Diego, Calif.) or tissue homogenate in a 384-well polystyrene assay plate (Corning, Corning, N.Y.).\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705852.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705852/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.92\n",
      "      max pchembl_value: 10.97\n",
      "      median pchembl_value: 7.82\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3706341:\n",
      "    description: \"Assay CHEMBL3706341: Alphascreen Assay: The PI3K AlphaScreen assay (PerkinElmer, Waltham, Mass.) measures the activity of a panel of four phosphoinositide 3-kinases: PI3Kalpha, PI3Kbeta, PI3Kgamma, and PI3Kdelta. Each of these enzymes phosphorylates the 3'-hydroxyl group on phosphatidylinositiol (4,5)-bisphosphate (PIP2) to produce phosphatidylinositol (3,4,5)-trisphosphate (PIP3). This phosphorylation activity is measured using a GST-tagged PIP3 binding protein (Echelon Biosciences, Salt Lake City, Utah), an anti-GST-tagged Acceptor bead, and streptavidin-Donor bead. The interaction of biotinylated-PIP3 analog (IP4) and the PIP3 binding protein brings both Acceptor and Donor beads together producing, upon excitation of the Donor beads at 680 nm, a singlet oxygen species leading to the luminescent AlphaScreen signal. When PIP3 is produced via phosphorylation of PIP2 by a PI3K, PIP3 competes with biotinylated-PIP3 analog (IP4) for binding to the PIP3 binding protein.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3706341.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3706341/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: Ki\n",
      "      min pchembl_value: 4.64\n",
      "      max pchembl_value: 9.00\n",
      "      median pchembl_value: 7.44\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL1267248:\n",
      "    description: \"Assay CHEMBL1267248: Antiplasmodial activity against Plasmodium falciparum Dd2 after 72 hrs by SYBR green assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL1267248.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL1267248/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.80\n",
      "      max pchembl_value: 9.89\n",
      "      median pchembl_value: 5.00\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3888977:\n",
      "    description: \"Assay CHEMBL3888977: Biochemical Assay: The FASN enzyme was isolated from SKBr3 cells. SKBr3 is a human breast cancer cell-line with high levels of FASN expression. It is estimated that FASN comprises about 25% of the cytosolic proteins in this cell line. SKBr3 cells were homogenized in a dounce homogenizer then centrifuged for 15 minutes at 4° C. to remove particulate matter. The supernatant was then analyzed for protein content, diluted to the appropriate concentration, and used to measure FASN activity. The presence of FASN was confirmed by western blot analysis. A similar method for isolation of FASN from SKBr3 cells is described in Teresa, P. et al. (Clin. Cancer Res. 2009; 15(24), 7608-7615). FASN activity of the SKBr3 cell extract was determined by measuring either NADPH oxidation or the amount of thiol-containing coenzyme A (CoA) released during the fatty acid synthase reaction. The dye CPM (7-diethylamino-3-(4'-maleimidyl-phenyl)-4-methylcoumarin) contains a thiol reactive group that increases its fluorescence emission on reaction with the sulfhydryl group of CoA.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3888977.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3888977/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.30\n",
      "      max pchembl_value: 8.30\n",
      "      median pchembl_value: 7.29\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3887759:\n",
      "    description: \"Assay CHEMBL3887759: Enzyme Assay: JAK1, JAK2, JAK3 and Tyk2 were purchased from Carna Biosciences, Inc. As the substrate, LANCE Ultra ULight-JAK1 Peptide (manufactured by PerkinElmer Co., Ltd. (PE)) was used. Dilute solutions of compounds and enzymes in assay buffer (50 mM HEPES pH7.5, 1 mM EGTA, 1 mM MgCl2, 2 mM DTT, 0.01% Tween20) were dispensed into wells of a 384-well black plate. After 5 minutes of preincubation, dilute solutions of the substrate and ATP (adenosine triphosphate) were added at a final concentration of 100 μM, and the plate was incubated at room temperature for 2 hours. After addition of a termination reagent containing EDTA (ethylenediamine tetraacetic acid) at a final concentration of 10 mM, LANCE Eu-W1024 Anti-phosphotyrosine (PT66) (manufactured by PE) was added, and after 1 hour of incubation, the fluorescences were measured with ARVO-HTS.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3887759.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3887759/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.28\n",
      "      max pchembl_value: 8.92\n",
      "      median pchembl_value: 6.57\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3888980:\n",
      "    description: \"Assay CHEMBL3888980: Functional Assay: The assay was performed as described below, using HEK293 cells transfected with rat TRPV1 (rTRPV1/HEK293). These cells had a geneticin selection marker and were grown in Dulbecco's Modified Eagle Medium (DMEM, Gibco BRL) supplemented with 10% fetal bovine serum, penicillin/streptomycin (50 units/mL), and 500 μg/mL geneticin in 5% CO2 at 37° C. Cells were passaged frequently, every 3-5 days, to avoid overgrowth, depletion of essential medium components, or acidic medium exposure. Cells were passaged using a brief wash in 0.05% trypsin with 1 mM EDTA, followed by dissociation in divalent-free phosphate-buffered saline (Hyclone #SH30028.02). Dissociated cells were seeded onto poly-D-lysine coated black-walled 96-well plates (Biocoat; Becton Dickinson #354640) at about 40,000 cells per well and grown for approximately 1 day in culture medium to near confluency. The assay buffer was composed of 130 mM NaCl, 2 mM KCl, 2 mM MgCl2, 10 mM HEPES, 5 mM glucose, and either 2 mM or 20 μM CaCl2. On the day of the experiment, the culture medium was replaced with 2 mM calcium assay buffer using an automated plate washer (ELx405; Biotek, VT). The cells were incubated in 100 μL/well Fluo-3/AM (2 μM; TEFLabs #0116) with Pluronic F127 (100 μg/mL; Sigma #P2443) for 1 h at rt in the dark. After loading the cells, the dye solution was replaced with 50 μL/well of 20 μM calcium assay buffer using the ELx405 plate washer. Test compounds (50 μL/well) were added to the plate and incubated for 30 min. Intracellular Ca2+ levels were subsequently assayed using a Fluorometric Imaging Plate Reader (FLIPR™ instrument, Molecular Devices, CA) to simultaneously monitor Fluo-3 fluorescence in all wells (λexcitation=488 nm, λemission=540 nm) during challenge with agonist (capsaicin). The IC50 values were determined. Cells were challenged with 150 nM capsaicin and the fluorescence counts were captured following agonist addition at a sampling rate of 0.33 Hz. The contents of the wells were mixed 3 times (40 μL mix volume) immediately after the additions were made. Concentration dependence of block was determined by exposing each well of cells in duplicate rows of a 96 well plate to a serial dilution of test compound. The concentration series usually started at 10 Î¼M with a three-fold serial decrement in concentration. The magnitude of the capsaicin response was determined by measuring the change in fluo3 fluorescence before and 100 seconds after the addition of the agonist.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3888980.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3888980/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.72\n",
      "      max pchembl_value: 8.40\n",
      "      median pchembl_value: 6.54\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705217:\n",
      "    description: \"Assay CHEMBL3705217: Scintillation Proximity Assay : PDE10 activity of the compounds of the present invention was determined using a Scintillation Proximity Assay (SPA)-based method similar to the one previously described (Fawcett, L. et al., Proc Natl Acad Sci USA (2000) 97(7):3702-3707). The human PDE10A full length assay was performed in 96-well micro titer plates. The reaction mixture of 50 μl contained 20 mM HEPES pH=7.5/10 mM MgCl2/0.05 mg/ml BSA (Sigma cat. # A-7906), 50 nM cGMP (Sigma, cat. # G6129) and 50 nM [3H]-cGMP (GE Healthcare, cat. # TRK392 S.A. 13.2 Ci/mmol), 3.75 ng/well PDE10A enzyme (Enzo Life Science, Lausen, Switzerland cat #SE-534) with or without a specific test compound.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705217.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705217/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.57\n",
      "      max pchembl_value: 10.00\n",
      "      median pchembl_value: 8.07\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3887318:\n",
      "    description: \"Assay CHEMBL3887318: Inhibition Assay: Stably expressing cell line (C6BU-1 cell transfected with human P2X3 receptor gene (GenBank accession number Y07683)) was used. The cells were seeded in a 96-well microtiter plate at a concentration of 8000 cells/well and cultured in the medium (8.3% fetal bovine serum, 8.3% horse serum, 1% antibiotic and antifungal in DMEM) for one day at 37 C. under 5% carbon dioxide atmosphere. The medium was replaced with 4 uM Fluo-3-AM solution (pH 7.5) containing 20 mM HEPES, 137 mM NaCl, 2.7 mM KCl, 0.9 mM MgCl2, 5.0 mM CaCl2, 5.6 mM D-glucose, 2.5 mM probenecid, 10% BSA, and 0.08% Pluronic F-127, and incubated at 37° C. under 5% dioxide carbon atmosphere for one hour. The plate was washed with washing buffer (20 mM HEPES, 137 mM NaCl, 2.7 mM KCl, 0.9 mM MgCl2, 5.0 mM CaCl2, 5.6 mM D-glucose, 2.5 mM probenecid, pH 7.5), and each well was added with 40 uL of this buffer. The plate was placed in High-Throughput Screening System FDSS 3000 (Hamamatsu Photonics K.K.).\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3887318.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3887318/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 6.00\n",
      "      max pchembl_value: 8.52\n",
      "      median pchembl_value: 7.24\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705924:\n",
      "    description: \"Assay CHEMBL3705924: Activation Assay: The cDNA for human TRPV1 (hTRPV1) was isolated by reverse transcriptase-polymerase chain reaction (RT-PCR) from human small intestine poly A+ RNA supplied by Clontech (Palo Alto, Calif.) using primers designed surrounding the initiation and termination codons identical to the published sequences (Hayes et al. Pain 2000, 88, 205-215). The resulting cDNA PCR products were subcloned into pCIneo mammalian expression vector (Promega) and fully sequenced using fluorescent dye-terminator reagents (Prism, PerkinElmer Applied Biosystems Division) and a PerkinElmer Applied Biosystems Model 373 DNA sequencer or Model 310 genetic analyzer. Expression plasmids encoding the hTRPV1 cDNA were transfected into HEK293 cells using Lipofectamine. Forty-eight hours after transfection, the neomycin-resistant cells were selected with growth medium containing 800 ug/mL Geneticin (Life Technologies, formerly Gibco BRL). Surviving individual colonies were isolated and screened for TRPV1 activity.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705924.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705924/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.51\n",
      "      max pchembl_value: 9.00\n",
      "      median pchembl_value: 6.96\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3887188:\n",
      "    description: \"Assay CHEMBL3887188: Functional Assay: HEK293 cells were transfected with human TRPV1 cloned in pcDNA3.1zeo(+) using the Effectene non-liposomal lipid based transfection kit (Qiagen) (hTRPV1/HEK293). hTRPV1/HEK293 cells were routinely grown as monolayers under selection in zeocin (200 μg/mL; Invitrogen) in Dulbecco's Modified Eagle Medium (DMEM, Gibco BRL) supplemented with 10% fetal bovine serum, and penicillin/streptomycin (50 units/mL) in 5% CO2 at 37° C. Cells were passaged frequently, every 3-5 days, to avoid overgrowth, depletion of essential medium components, or acidic medium exposure. Cells were passaged using a brief wash in 0.05% trypsin with 1 mM EDTA, followed by dissociation in divalent-free phosphate-buffered saline (Hyclone #SH30028.02). Dissociated cells were seeded onto poly-D-lysine coated black-walled 96-well plates (Biocoat; Becton Dickinson #354640) at about 40,000 cells per well and grown for approximately 1 day in culture medium to near confluency. The assay buffer was composed of 130 mM NaCl, 2 mM KCl, 2 mM MgCl2, 10 mM HEPES, 5 mM glucose, and either 2 mM or 20 μM CaCl2. On the day of the experiment, the culture medium was replaced with 2 mM calcium assay buffer using an automated plate washer (ELx405; Biotek, VT). The cells were incubated in 100 μL/well Fluo-3/AM (2 μM; TEFLabs #0116) with Pluronic F127 (100 μg/mL; Sigma #P2443) for 1 h at rt in the dark. After loading the cells, the dye solution was replaced with 50 μL/well of 20 μM calcium assay buffer using the ELx405 plate washer. Test compounds (50 μL/well) were added to the plate and incubated for 30 min. Intracellular Ca2+ levels were subsequently assayed using a Fluorometric Imaging Plate Reader (FLIPR™ instrument, Molecular Devices, CA) to simultaneously monitor Fluo-3 fluorescence in all wells (λexcitation=488 nm, λemission=540 nm) during challenge with agonist (capsaicin). The IC50 values were determined. Cells were challenged with 150 nM capsaicin and the fluorescence counts were captured following agonist addition at a sampling rate of 0.33 Hz. The contents of the wells were mixed 3 times (40 μL mix volume) immediately after the additions were made. Concentration dependence of block was determined by exposing each well of cells in duplicate rows of a 96 well plate to a serial dilution of test compound. The concentration series usually started at 10 Î¼M with a three-fold serial decrement in concentration. The magnitude of the capsaicin response was determined by measuring the change in fluo3 fluorescence before and 100 seconds after the addition of the agonist.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3887188.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3887188/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.85\n",
      "      max pchembl_value: 8.70\n",
      "      median pchembl_value: 6.84\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3888536:\n",
      "    description: \"Assay CHEMBL3888536: Homogeneous Time Resolved Fluorescence Assay: The sGC cellular activator assay is performed in the presence and absence of 50% human serum (HS) using Chinese hamster ovary cells that have been stably transfected to express the human soluble guanylate cyclase alpha 1 and beta 1 subunits (sGC). Cells are preincubated with 40 microM 1H-[1,2,4]oxadiazolo[4,3-a]quinoxalin-1-one (ODQ), an sGC inhibitor, for one h in buffer containing 0.1% bovine serum albumin and 3-isobutyl-1-methylxanthine (IBMX). Concentration response curves are prepared for test compounds in DMSO. An intermediate dilution of the compounds is performed in either buffer containing IBMX or type AB HS containing IBMX. Diluted compounds are added to cells and they are incubated at room temperature for thirty min. cGMP is measured using a CisBio homogeneous time resolved fluorescence kit and the EC50 is calculated for each compound.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3888536.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3888536/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: EC50\n",
      "      min pchembl_value: 6.13\n",
      "      max pchembl_value: 9.00\n",
      "      median pchembl_value: 7.85\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3707951:\n",
      "    description: \"Assay CHEMBL3707951: In Vitro Enzyme Assay: The ability of compounds to inhibit the kinase activity of baculovirus-expressed human FAK and JAK kinase using the time-resolved fluorescence (TRF) assay system.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3707951.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3707951/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.00\n",
      "      max pchembl_value: 8.82\n",
      "      median pchembl_value: 6.48\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3707962:\n",
      "    description: \"Assay CHEMBL3707962: Biological Assay: Endothelial lipase (EL) and hepatic lipase (HL) activities were measured using a fluorescent substrate, A10070, (Invitrogen, CA) doped into an artificial vesicle containing DMPG (Avanti Polar Lipids) as the excipient. Vesicles were prepared by combining 571 uL of 29 mM DMPG in a 1:1 mixture of MeOH and CHCl3 with 2000 uL of 1 mM A10070 in a 1:1 mixture of MeOH and CHCl3. The mixture was dried under nitrogen in multiple vials then resuspended in 20 mL total volume of 50 mM HEPES pH 8.0 buffer containing 50 mM NaCl and 0.2 mM EDTA. The sample was allowed to sit at room temperature for 15 min and then was sonicated 3x4 mins on ice with a Branson Sonicator using duty cycle 1. This preparation provides vesicles with a mole fraction of 0.11 for the FRET substrate.The enzymatic assay was measured using 384-well white Optiplates. Each well contained 20 uL of assay buffer (50 mM HEPES pH 8.0, 50 mM NaCl and 1 mM CaCl2) and 0.25 uL of a DMSO solution containing a compound.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3707962.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3707962/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.60\n",
      "      max pchembl_value: 9.30\n",
      "      median pchembl_value: 7.02\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3889140:\n",
      "    description: \"Assay CHEMBL3889140: Radioligand Binding Assay: HEK-293 cells stably expressing rat TAAR1 were maintained at 37° C. and 5% CO2 in DMEM high glucose medium, containing fetal calf serum (10%, heat inactivated for 30 min at 56° C.), penicillin/streptomycin (1%), and 375 ug/ml geneticin (Gibco). Cells were released from culture flasks using trypsin/EDTA, harvested, washed twice with ice-cold PBS (without Ca2+ and Mg2+), pelleted at 1,000 rpm for 5 min at 4° C., frozen and stored at -80° C. Frozen pellets were suspended in 20 ml HEPES-NaOH (20 mM, pH 7.4) containing 10 mM EDTA and homogenized with a Polytron (PT 6000, Kinematica) at 14,000 rpm for 20 s. The homogenate was centrifuged at 48,000Ã¿g for 30 min at 4° C. Subsequently, the supernatant was removed and discarded, and the pellet resuspended in 20 ml HEPES-NaOH (20 mM, pH 7.4) containing 0.1 mM EDTA using the Polytron (20 s at 14,000 rpm). This procedure was repeated and the final pellet resuspended in HEPES-NaOH containing 0.1 mM EDTA and homogenized using the Polytron. Typically, aliquots of 2 ml membrane portions were stored at -80° C. With each new membrane batch the dissociation constant (Kd) was determined via a saturation curve. The TAAR1 radioligand 3[H]â¿¿(S)-4-[(ethyl-phenyl-amino)-methyl]-4,5-dihydro-oxazol-2-ylamine (described in WO 2008/098857) was used at a concentration equal to the calculated Kd value, that was usually around 2.3 nM, resulting in the binding of approximately 0.2% of the radioligand and a specific binding representing approximately 85% of the total binding. Nonspecific binding was defined as the amount of 3[H]â¿¿(S)-4-[(ethyl-phenyl-amino)-methyl]-4,5-dihydro-oxazol-2-ylamine bound in the presence of 10 uM unlabeled ligand. All compounds were tested at a broad range of concentrations (10 pM to 10 uM) in duplicates. The test compounds (20 ul/well) were transferred into a 96 deep well plate (TreffLab), and 180 ul of HEPES-NaOH (20 mM, pH 7.4) containing MgCl2 (10 mM) and CaCl2 (2 mM) (binding buffer), 300 ul of the radioligand 3[H]â¿¿(S)-4-[(ethyl-phenyl-amino)-methyl]-4,5-dihydro-oxazol-2-ylamine at a concentration of 3.3xKd in nM and 500 ul of the membranes (resuspended at 50 ug protein per ml) added. The 96 deep well plates were incubated for 1 hr at 4° C. Incubations were terminated by rapid filtration through Unifilter-96 plates (Packard Instrument Company) and glass filters GF/C (Perkin Elmer) presoaked for 1 hr in polyethylenimine (0.3%) and washed 3 times with 1 ml of cold binding buffer. After addition of 45 ul of Microscint 40 (PerkinElmer) the Unifilter-96 plate was sealed and after 1 hr the ratioactivity counted using a TopCount Microplate Scintillation Counter (Packard Instrument Company).\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3889140.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3889140/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: Ki\n",
      "      min pchembl_value: 5.53\n",
      "      max pchembl_value: 9.70\n",
      "      median pchembl_value: 8.21\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705282:\n",
      "    description: \"Assay CHEMBL3705282: Enzyme Inhibition Assay: Enzyme activity: An IMAP TR-FRET assay was used to analyze the enzyme activity (Molecular Devices Corp., Sunnyvale, Calif).\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705282.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705282/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.24\n",
      "      max pchembl_value: 9.70\n",
      "      median pchembl_value: 6.78\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705630:\n",
      "    description: \"Assay CHEMBL3705630: Enzyme Assay-Time Dependence: Compounds were assessed for their ability to inhibit the enzymatic activity of a recombinant form of Glutaminase 1 (GAC) using a biochemical assay that couples the production of glutamate (liberated by GAC) to glutamate dehydrogenase (GDH) and measuring the change in absorbance for the reduction of NAD+ to NADH.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705630.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705630/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.50\n",
      "      max pchembl_value: 8.70\n",
      "      median pchembl_value: 8.15\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705791:\n",
      "    description: \"Assay CHEMBL3705791: Fluorescence Biochemical Assay: The IDH1 (R132H) mutant catalyzes the reduced form of NADP+ (NADPH) and α-ketoglutarate (α-KG) to form nicotinamide adenine dinucleotide phosphate (NADP+) and R (−)-2-hydroxyglutarate (2HG). The reaction can be monitored kinetically by following the oxidation of NADPH to NADP+ which is measured using fluorescence, excitation at 355 nm and emission at 530 nm. Reactions were monitored using the Perkin-Elmer Envision, Model 2101. More specifically, the biochemical reactions were performed at room temperature in 384-well Greiner flat-bottom plates (Cat. No. 781076) using a final reaction volume of 20 μL and the following assay buffer conditions: 50 mM HEPES pH 7.5, 10 mM MgCl2, 1 mM DTT, 0.02% BSA, 0.02% Tween-20, 10 μM NADPH and 100 μM α-KG. The final reaction mixture contained 2.5% DMSO and test compounds with concentrations ranging 0.0000008-25 μM. The IDH1 (R132H) enzyme was used at a final concentration of 10 nM.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705791.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705791/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.60\n",
      "      max pchembl_value: 8.12\n",
      "      median pchembl_value: 6.48\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3721139:\n",
      "    description: \"Assay CHEMBL3721139: Antagonist activity against human PAR4 expressed in HEK293 cells assessed as reduction in H-Ala-Phe(4-F)-Pro-Gly-Trp-Leu-Val-Lys-Asn-Gly-NH2-induced intracellular calcium mobilization incubated for 30 mins by FLIPR assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3721139.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3721139/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: EC50\n",
      "      min pchembl_value: 5.46\n",
      "      max pchembl_value: 9.74\n",
      "      median pchembl_value: 9.09\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705828:\n",
      "    description: \"Assay CHEMBL3705828: Homogeneous Time Resolved Fluorescence Assay: The assay below is used to test the modulatory activity of compounds against FLAP. Human and mouse FLAP-encoding DNA was amplified by polymerase chain reaction and cloned into pFastBac1 (Invitrogen) with a NH2-terminal 6-His tag for expression in Spodoptera frugiperda (Sf-9) cells. FLAP-containing membranes were prepared as was a FITC-labeled FLAP modulator (3-(3-(tert-butylthio)-1-(4-chlorobenzyl)-5-(quinolin-2-ylmethoxy)-1H-indol-2-yl)-2,2-dimethylpropanoic acid). The FLAP binding assay is performed in HTRF format (homogeneous time resolved fluorescence). FLAP-containing membranes (1 ug/well final for human) are incubated in the presence of the HTRF ligand, [5-[({[2-(2-{3-[3-(tert-butylsulfanyl)-1-(4-chlorobenzyl)-5-(quinolin-2-ylmethoxy)-1H-indol-2-yl]-2,2-dimethylpropanoyl}hydrazino)-2-oxoethyl]sulfanyl}acetyl)amino]-2-(6-hydroxy-3-oxo-3H-xanthen-9-yl)benzoic acid] (25 nM final), a terbium labeled anti-His tag antibody (0.5 ng/well final, from Cisbio) and compounds.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705828.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705828/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: Ki\n",
      "      min pchembl_value: 5.60\n",
      "      max pchembl_value: 9.40\n",
      "      median pchembl_value: 7.33\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3705464:\n",
      "    description: \"Assay CHEMBL3705464: Time Resolved Fluorescence Energy Transfer (TR-FRET) Assay: The inhibition of p53-MDM2 and p53-MDM4 interactions is measured by time resolved fluorescence energy transfer (TR-FRET). Fluorescence energy transfer (or Foerster resonance energy transfer) describes an energy transfer between donor and acceptor fluorescent molecules. For this assay, human MDM2 protein (amino acids 2-188) and human MDM4 protein (amino acids 2-185), tagged with a C-terminal biotin moiety, are used in combination with a Europium labeled streptavidin (Perkin Elmer, Inc., Waltham, Mass., USA) serving as the donor fluorophore. The p53 derived, Cy5 labeled peptide Cy5-TFSDLWKLL (p53 aa18-26) is the energy acceptor. Upon excitation of the donor molecule at 340 nm, binding interaction between MDM2 or MDM4 and the p53 peptide induces energy transfer and enhanced response at the acceptor emission wavelength at 665 nm.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3705464.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3705464/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 6.30\n",
      "      max pchembl_value: 10.40\n",
      "      median pchembl_value: 9.44\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3888799:\n",
      "    description: \"Assay CHEMBL3888799: Homogeneous Time Resolved Fluorescence Assay (HTRF): The assay for the determination of Pim activity is based on the formation of phosphorylated biotinylated-BAD peptide at the Serine 112 residue (S112) and employs HTRF® (homogeneous time resolved fluorescence) technology to detect the product in a 384-well plate format. The phosphorylation of biotinylated-BAD (S112) peptide by full length recombinant Pim-1, Pim-2, or Pim-3 protein was detected with streptavidin:Allophycocyanin (APC) conjugate and a europium (Eu) labeled antibody directed against phosphorylated-BAD (S112). Excitation of Eu by a high energy laser light (337 nm) leads to a transfer of energy to the APC molecule, and results in an emission at 665 nm. The fluorescence is directly proportional to the amount of phosphorylated BAD peptide present in the reaction. Compounds were prepared in DMSO by conducting 3-fold serial dilutions to give a 22-point dosing curve having a high dose of 1 μM. A reference compound was included on each assay plate [Costar 3658] in order to validate that plate; on one plate of every assay run, two additional reference compounds were included. The Reaction Buffer consisted of 45 mM Hepes, pH 7.0, 15 mM NaCl, and 1 mM MgCl. The quench/detection buffer consisted of 50 mM Tris, 100 mM NaCl, 0.05% BSA, 0.1% Tween and 3 mM EDTA. Biotinylated BAD peptide (Biopeptide), 10 mM ATP (Sigma), Labeled p-BAD (S112) mAb (Cell Signalling and Perkin Elmer) [with 0.05% BSA and 2 mM DTT added] streptavidin:Allophycocyanin [Perkin Elmer]. Final concentrations either Pim-1 enzyme [5 pM], or Pim-2 enzyme [0.5 pM], DMSO [1%], BLC BAD (S112) [0.5 μM], ATP [1.5 μM], streptavidin:Allophycocyanin [0.002 mg/mL] and biotinylated-BAD (S112) mAb [100 pM]. Initial incubations were carried out at RT (22° C.) for 30 min for both Pim-1 and for Pim-2. Pim enzyme is added to compound in buffer, and plates are incubated of 30 min. Biotinylated BAD and ATP are added and plates are incubated for 1 h. A mixture of labeled p-BAD (S112) mAb and quench/detection buffer are added and incubated for 2 h. Fluorescence was measured by an HTRF® Envision microplate reader.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3888799.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3888799/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.95\n",
      "      max pchembl_value: 10.92\n",
      "      median pchembl_value: 8.63\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3888094:\n",
      "    description: \"Assay CHEMBL3888094: Fluorescence Polarization Assays: In another embodiment, Fluorescence Polarization (FP) based assays may be used to detect and monitor ligand binding. Fluorescence polarization is a versatile laboratory technique for measuring equilibrium binding, nucleic acid hybridization, and enzymatic activity. Fluorescence polarization assays are homogeneous in that they do not require a separation step such as centrifugation, filtration, chromatography, precipitation, or electrophoresis. These assays are done in real time, directly in solution and do not require an immobilized phase. Polarization values can be measured repeatedly and after the addition of reagents since measuring the polarization is rapid and does not destroy the sample. Generally, this technique can be used to measure polarization values of fluorophores from low picomolar to micromolar levels.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3888094.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3888094/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.74\n",
      "      max pchembl_value: 8.00\n",
      "      median pchembl_value: 6.40\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3888800:\n",
      "    description: \"Assay CHEMBL3888800: Homogeneous Time Resolved Fluorescence Assay (HTRF): The assay for the determination of Pim activity is based on the formation of phosphorylated biotinylated-BAD peptide at the Serine 112 residue (S112) and employs HTRF® (homogeneous time resolved fluorescence) technology to detect the product in a 384-well plate format. The phosphorylation of biotinylated-BAD (S112) peptide by full length recombinant Pim-1, Pim-2, or Pim-3 protein was detected with streptavidin:Allophycocyanin (APC) conjugate and a europium (Eu) labeled antibody directed against phosphorylated-BAD (S112). Excitation of Eu by a high energy laser light (337 nm) leads to a transfer of energy to the APC molecule, and results in an emission at 665 nm. The fluorescence is directly proportional to the amount of phosphorylated BAD peptide present in the reaction. Compounds were prepared in DMSO by conducting 3-fold serial dilutions to give a 22-point dosing curve having a high dose of 1 μM. A reference compound was included on each assay plate [Costar 3658] in order to validate that plate; on one plate of every assay run, two additional reference compounds were included. The Reaction Buffer consisted of 45 mM Hepes, pH 7.0, 15 mM NaCl, and 1 mM MgCl. The quench/detection buffer consisted of 50 mM Tris, 100 mM NaCl, 0.05% BSA, 0.1% Tween and 3 mM EDTA. Biotinylated BAD peptide (Biopeptide), 10 mM ATP (Sigma), Labeled p-BAD (S112) mAb (Cell Signalling and Perkin Elmer) [with 0.05% BSA and 2 mM DTT added] streptavidin:Allophycocyanin [Perkin Elmer]. Final concentrations either Pim-1 enzyme [5 pM], or Pim-2 enzyme [0.5 pM], DMSO [1%], BLC BAD (S112) [0.5 μM], ATP [1.5 μM], streptavidin:Allophycocyanin [0.002 mg/mL] and biotinylated-BAD (S112) mAb [100 pM]. Initial incubations were carried out at RT (22° C.) for 30 min for both Pim-1 and for Pim-2. Pim enzyme is added to compound in buffer, and plates are incubated of 30 min. Biotinylated BAD and ATP are added and plates are incubated for 1 h. A mixture of labeled p-BAD (S112) mAb and quench/detection buffer are added and incubated for 2 h. Fluorescence was measured by an HTRF® Envision microplate reader.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3888800.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3888800/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.77\n",
      "      max pchembl_value: 10.92\n",
      "      median pchembl_value: 8.55\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL763870:\n",
      "    description: \"Assay CHEMBL763870: Inhibition of human Prostaglandin G/H synthase 2\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL763870.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL763870/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.02\n",
      "      max pchembl_value: 9.00\n",
      "      median pchembl_value: 7.11\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3888435:\n",
      "    description: \"Assay CHEMBL3888435: Cell-Based Assay: Compounds were tested on human Nav1.7 (hNav1.7) expressed stably in 293 cells, which have no endogenous sodium channels. Two forms of whole-cell patch-clamp assays were used. In both, peak current through Nav1.7 was evoked by a depolarizing voltage pulse, and currents were monitored as a function of time in response to exposure to several different concentrations of drug. Automated electrophysiology uses the PatchXpress 7000A system. Up to sixteen cells at a time were voltage-clamped in parallel, and the steady-state voltage dependence of inactivation was determined experimentally for each cell. For each cell, fractional inactivation was set at 20%, and peak current through Nav1.7 as a function of time was measured in response to increasing concentrations of test compound. Manual electrophysiology refers to conventional whole-cell patch-clamping (Hamill et al., 1981). After formation of the whole-cell clamp, cells were lifted off the bottom of the dish with the patch pipette and positioned directly in front of an array of glass microperfusion tubes each with internal diameter about 250 microns, with individual compound concentrations flowing from each tube. Current as a function of time in response to increasing drug concentrations was measured with Nav1.7 fully in the resting state (holding voltage set to −140 mV) and on the same cells with the holding voltage set to produce about 20% inactivation. Compounds were not applied for a prespecified time, but for as long as required to achieve a stable level of block.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3888435.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3888435/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.54\n",
      "      max pchembl_value: 7.10\n",
      "      median pchembl_value: 5.56\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3889139:\n",
      "    description: \"Assay CHEMBL3889139: Radioligand Binding Assay: HEK-293 cells stably expressing mouse TAAR1 were maintained at 37° C. and 5% CO2 in DMEM high glucose medium, containing fetal calf serum (10%, heat inactivated for 30 min at 56° C.), penicillin/streptomycin (1%), and 375 ug/ml geneticin (Gibco). Cells were released from culture flasks using trypsin/EDTA, harvested, washed twice with ice-cold PBS (without Ca2+ and Mg2+), pelleted at 1,000 rpm for 5 min at 4° C., frozen and stored at -80° C. Frozen pellets were suspended in 20 ml HEPES-NaOH (20 mM, pH 7.4) containing 10 mM EDTA and homogenized with a Polytron (PT 6000, Kinematica) at 14,000 rpm for 20 s. The homogenate was centrifuged at 48,000Ã¿g for 30 min at 4° C. Subsequently, the supernatant was removed and discarded, and the pellet resuspended in 20 ml HEPES-NaOH (20 mM, pH 7.4) containing 0.1 mM EDTA using the Polytron (20 s at 14,000 rpm). This procedure was repeated and the final pellet resuspended in HEPES-NaOH containing 0.1 mM EDTA and homogenized using the Polytron. Typically, aliquots of 2 ml membrane portions were stored at -80° C. With each new membrane batch the dissociation constant (Kd) was determined via a saturation curve. The TAAR1 radioligand 3[H]â¿¿(S)-4-[(ethyl-phenyl-amino)-methyl]-4,5-dihydro-oxazol-2-ylamine (described in WO 2008/098857) was used at a concentration equal to the calculated Kd value, that was usually around 2.3 nM, resulting in the binding of approximately 0.2% of the radioligand and a specific binding representing approximately 85% of the total binding. Nonspecific binding was defined as the amount of 3[H]â¿¿(S)-4-[(ethyl-phenyl-amino)-methyl]-4,5-dihydro-oxazol-2-ylamine bound in the presence of 10 uM unlabeled ligand. All compounds were tested at a broad range of concentrations (10 pM to 10 uM) in duplicates. The test compounds (20 ul/well) were transferred into a 96 deep well plate (TreffLab), and 180 ul of HEPES-NaOH (20 mM, pH 7.4) containing MgCl2 (10 mM) and CaCl2 (2 mM) (binding buffer), 300 ul of the radioligand 3[H]â¿¿(S)-4-[(ethyl-phenyl-amino)-methyl]-4,5-dihydro-oxazol-2-ylamine at a concentration of 3.3xKd in nM and 500 ul of the membranes (resuspended at 50 ug protein per ml) added. The 96 deep well plates were incubated for 1 hr at 4° C. Incubations were terminated by rapid filtration through Unifilter-96 plates (Packard Instrument Company) and glass filters GF/C (Perkin Elmer) presoaked for 1 hr in polyethylenimine (0.3%) and washed 3 times with 1 ml of cold binding buffer. After addition of 45 ul of Microscint 40 (PerkinElmer) the Unifilter-96 plate was sealed and after 1 hr the ratioactivity counted using a TopCount Microplate Scintillation Counter (Packard Instrument Company).\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3889139.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3889139/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: Ki\n",
      "      min pchembl_value: 5.00\n",
      "      max pchembl_value: 9.52\n",
      "      median pchembl_value: 7.64\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3889083:\n",
      "    description: \"Assay CHEMBL3889083: LC-MS Detection of 2-HG Assay: The biochemical reactions were performed at room temperature in 384-well Greiner flat-bottom plates (Costar, Cat. No. 781201) using a final reaction volume of 30 μL and the following assay buffer conditions: 50 mM HEPES pH 7.4, 10 mM MgCl2, 50 mM KCl, 1 mM DTT, 0.02% BSA, 5 uM NADPH and 100 uM alpha-KG. The final reaction mixture contained 3.3% DMSO and inhibitors with concentrations ranging 0.02-50 μM. The IDH1 enzyme was used at a final concentration of 0.25 nM. Following 45 minutes incubation, the reaction mixtures were quenched by the addition of 10 μL of 16% formic acid containing 800 nM of 5-carbon labeled 13C-2-HG). The protein was then precipitated by the addition of 2.5 volumes of acetonitrile followed by centrifugation (3000×g, minutes). The concentration of 2-HG in the resulting supernatants was measured by LC-MS (see below). LC-MS method. Reaction mixture supernatants were submitted to chromatographic separation on a BiobasicAX column (2.1 mm×20 mm, 5 μm particle, Thermo Scientific Inc.). The chromatographic mobile phases were A) 25 mM ammonium biocarbonate and B) acetonitrile (0.1% ammonium hydroxide). Nicotinamide was eluted at 1 ml/min using a 85-5% B gradient over 0.9 minutes (Agilent 1200SL LC system, Thermofisher LX-4 autosampler) and analyzed by multiple reaction monitoring (MRM) on a API4000 QTrap mass spectrometer (ABSciex, Framingham, Mass.) in the positive electrospray ionization (ESI+) mode. The mass transition for 2-HG and 13C-2-HG were 147→129 and 152→134, respectively.\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3889083.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3889083/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 4.33\n",
      "      max pchembl_value: 8.40\n",
      "      median pchembl_value: 6.80\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n",
      "  CHEMBL3734165:\n",
      "    description: \"Assay CHEMBL3734165: Inhibition of recombinant full-length human PKC-iota expressed in Sf21 cells using FAM-PKCs pseudopeptide derived peptide substrate and 150 uM ATP by fluorescence polarization based IMAP assay\"\n",
      "    args:\n",
      "      filename: '{{ CATALOG_DIR }}/source_data/assay_CHEMBL3734165.csv.gz'\n",
      "      smilesColumn: canonical_smiles\n",
      "    metadata:\n",
      "      url: https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL3734165/\n",
      "      source: ChEMBL_30\n",
      "      activity_type: IC50\n",
      "      min pchembl_value: 5.00\n",
      "      max pchembl_value: 7.00\n",
      "      median pchembl_value: 6.00\n",
      "    driver: intake_rdkit.smiles.SmilesSource\n",
      "\n"
     ]
    }
   ],
   "source": [
    "yaml=[]\n",
    "for aid,achemblid in zip(aids,achemblids):\n",
    "    d = %sql postgresql://localhost/chembl_30 \\\n",
    "select distinct on (cil.chembl_id) cil.chembl_id compound_chembl_id,canonical_smiles, \\\n",
    "        standard_value,standard_units,standard_relation,standard_type,pchembl_value \\\n",
    "          from activities join assays using(assay_id) \\\n",
    "          join compound_structures using(molregno) \\\n",
    "          join chembl_id_lookup cil on (molregno=entity_id and entity_type='COMPOUND') \\\n",
    "          where pchembl_value is not null  \\\n",
    "            and standard_value is not null and standard_units = 'nM' \\\n",
    "            and data_validity_comment is null \\\n",
    "            and assay_id=:aid \\\n",
    "          order by cil.chembl_id;\n",
    "    df = d.DataFrame()\n",
    "    df.to_csv(f'./datasets/source_data/assay_{achemblid}.csv.gz',index=False)\n",
    "    \n",
    "    minAct = min(df.pchembl_value)\n",
    "    maxAct = max(df.pchembl_value)\n",
    "    medAct = np.median(df.pchembl_value)\n",
    "    actTypes = list(set(df.standard_type))\n",
    "    if len(actTypes)>1:\n",
    "        print(f'SKIPPING {achemblid} since it has {len(actTypes)} different activity types.')\n",
    "        continue\n",
    "    actType = actTypes[0]\n",
    "    assayd = %sql postgresql://localhost/chembl_30 \\\n",
    "    select * from assays  \\\n",
    "        where assay_id=:aid\n",
    "    assayd = dict(assayd[0])\n",
    "    # quotes in the description cause problems with the yaml parsing\n",
    "    assayd['description'] = assayd['description'].replace('\"','')\n",
    "    template=f'''  {achemblid}:\n",
    "    description: \"Assay {achemblid}: {assayd['description']}\"\n",
    "    args:\n",
    "      filename: '{{{{ CATALOG_DIR }}}}/source_data/assay_{achemblid}.csv.gz'\n",
    "      smilesColumn: canonical_smiles\n",
    "    metadata:\n",
    "      url: https://www.ebi.ac.uk/chembl/assay_report_card/{achemblid}/\n",
    "      source: ChEMBL_30\n",
    "      activity_type: {actType}\n",
    "      min pchembl_value: {minAct:.2f}\n",
    "      max pchembl_value: {maxAct:.2f}\n",
    "      median pchembl_value: {medAct:.2f}\n",
    "    driver: intake_rdkit.smiles.SmilesSource\n",
    "'''\n",
    "    yaml.append(template)\n",
    "print('\\n'.join(yaml))\n",
    "with open('./datasets/assays.yaml','w+') as outf:\n",
    "    header='''metadata:\n",
    "  version: 1\n",
    "  creator: \n",
    "    name: greg landrum\n",
    "    email: glandrum@ethz.ch\n",
    "\n",
    "  summary: |\n",
    "    Collection of datasets with pchembl_values for bioactivity prediction.\n",
    "\n",
    "    Each row includes the reported value. Only values without data_validity_comments are included\n",
    "    \n",
    "sources:'''\n",
    "    print(header,file=outf)\n",
    "    print('\\n'.join(yaml),file=outf)\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.13"
  },
  "toc": {
   "base_numbering": 1,
   "nav_menu": {},
   "number_sections": true,
   "sideBar": true,
   "skip_h1_title": false,
   "title_cell": "Table of Contents",
   "title_sidebar": "Contents",
   "toc_cell": false,
   "toc_position": {},
   "toc_section_display": true,
   "toc_window_display": true
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
